<!DOCTYPE html>
<html lang="pt-BR"><head>
<script src="bioinfo-nanopore_files/libs/clipboard/clipboard.min.js"></script>
<script src="bioinfo-nanopore_files/libs/quarto-html/tabby.min.js"></script>
<script src="bioinfo-nanopore_files/libs/quarto-html/popper.min.js"></script>
<script src="bioinfo-nanopore_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="bioinfo-nanopore_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="bioinfo-nanopore_files/libs/quarto-html/light-border.css" rel="stylesheet">
<link href="bioinfo-nanopore_files/libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.6.39">

  <meta name="author" content="Marcel Ferreira">
  <meta name="dcterms.date" content="2025-09-27">
  <title>Sequenciamento de Terceira Geração com Oxford Nanopore</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="bioinfo-nanopore_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="bioinfo-nanopore_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for syntax highlighting */
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; font-weight: bold; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
  </style>
  <link rel="stylesheet" href="bioinfo-nanopore_files/libs/revealjs/dist/theme/quarto-423d414d36ce366a9d05f36d06304417.css">
  <link href="bioinfo-nanopore_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-pointer/pointer.css" rel="stylesheet">
  <link href="bioinfo-nanopore_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Sequenciamento de Terceira Geração com Oxford Nanopore</h1>
  <p class="subtitle">Bioinformática</p>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
Marcel Ferreira 
</div>
</div>
</div>

  <p class="date">2025-09-27</p>
</section>
<section id="bioinformática" class="slide level2">
<h2>Bioinformática</h2>
<ul>
<li>NIH: “<em>Bioinformatics, as related to genetics and genomics, is a scientific subdiscipline that involves using computer technology to collect, store, analyze and disseminate biological data and information, such as DNA and amino acid sequences or annotations about those sequences.</em>”</li>
</ul>

<aside><div>
<p>NIH: National Institute of Health</p>
</div></aside></section>
<section id="bioinformática-1" class="slide level2">
<h2>Bioinformática</h2>
<ul>
<li>NIH: “<em>Bioinformatics, as related to <strong>genetics</strong> and <strong>genomics</strong>, is a scientific subdiscipline that involves using computer technology to <strong>collect</strong>, <strong>store</strong>, <strong>analyze</strong> and <strong>disseminate</strong> biological data and information, such as <strong>DNA</strong> and <strong>amino acid sequences</strong> or annotations about those sequences.</em>”</li>
</ul>

<aside><div>
<p>NIH: National Institute of Health</p>
</div></aside></section>
<section id="oxford-nanopore" class="slide level2">
<h2>Oxford Nanopore</h2>

<img data-src="images/ont_prinmods.png" width="1250" class="r-stretch quarto-figure-center"><p class="caption">Principio do sequenciamento ONT</p></section>
<section id="oxford-nanopore-1" class="slide level2">
<h2>Oxford Nanopore</h2>
</section>
<section id="fluxograma" class="slide level2">
<h2>Fluxograma</h2>

<img data-src="Mermaid%20Chart%20-%20Create%20complex,%20visual%20diagrams%20with%20text.%20A%20smarter%20way%20of%20creating%20diagrams.-2025-07-23-163425.svg" width="1150" class="r-stretch quarto-figure-center"><p class="caption">Bioinformatics Pipeline</p></section>
<section id="basecalling" class="slide level2">
<h2>Basecalling</h2>
<ul>
<li><p>Procedimento no qual o sinal elétrico (<code>.pod5</code>) é convertido em bases (<code>.fastq</code> | <code>.bam</code>);</p></li>
<li><p>Momento crucial onde se define o perfil da análise que será realizada;</p></li>
<li><p>Erros aqui prejudicam todas as demais análises;</p></li>
<li><p><code>Dorado</code>;</p></li>
</ul>
</section>
<section id="basecalling---dorado" class="slide level2">
<h2>Basecalling - <code>Dorado</code></h2>
<ul>
<li>Contem diversos modelos:
<ul>
<li>chamada de bases;</li>
<li>modificação de bases;</li>
<li>polimento;</li>
<li>variantes;</li>
</ul></li>
</ul>
</section>
<section id="comando-dorado" class="slide level2">
<h2>Comando <code>dorado</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb1-1"><a href=""></a><span class="op">&gt;</span> dorado <span class="ex">--help</span></span>
<span id="cb1-2"><a href=""></a><span class="ex">Usage:</span> dorado <span class="pp">[</span><span class="ss">options</span><span class="pp">]</span> subcommand</span>
<span id="cb1-3"><a href=""></a></span>
<span id="cb1-4"><a href=""></a><span class="ex">Positional</span> arguments:</span>
<span id="cb1-5"><a href=""></a><span class="ex">aligner</span></span>
<span id="cb1-6"><a href=""></a><span class="ex">basecaller</span></span>
<span id="cb1-7"><a href=""></a><span class="ex">correct</span></span>
<span id="cb1-8"><a href=""></a><span class="ex">demux</span></span>
<span id="cb1-9"><a href=""></a><span class="ex">download</span></span>
<span id="cb1-10"><a href=""></a><span class="ex">duplex</span></span>
<span id="cb1-11"><a href=""></a><span class="ex">summary</span></span>
<span id="cb1-12"><a href=""></a><span class="ex">trim</span></span>
<span id="cb1-13"><a href=""></a></span>
<span id="cb1-14"><a href=""></a><span class="ex">Optional</span> arguments:</span>
<span id="cb1-15"><a href=""></a><span class="ex">-h</span> <span class="at">--help</span>               shows help message and exits</span>
<span id="cb1-16"><a href=""></a><span class="ex">-v</span> <span class="at">--version</span>            prints version information and exits</span>
<span id="cb1-17"><a href=""></a><span class="ex">-vv</span>                     prints verbose version information and exits</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="qual-modelo-usar" class="slide level2">
<h2>Qual modelo usar?</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb2-1"><a href=""></a><span class="ex">{analyte}_{pore}_{chemistry}_{speed}@version</span></span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li><p><code>analyte</code>: DNA ou RNA?</p></li>
<li><p><code>pore</code>: Qual técnologia de poro? (<code>R10.4.1</code>/<code>R9.4.1</code>)</p></li>
<li><p><code>chemistry</code>: Qual kit foi utilizado foi utilizado?</p></li>
<li><p><code>speed</code>: Velocidade de leitura durante o sequenciamento (DNA 400bps)</p></li>
<li><p><code>version</code>: Versão do modelo</p></li>
</ul>
</section>
<section id="modelos-dorado" class="slide level2">
<h2>Modelos <code>dorado</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb3-1"><a href=""></a><span class="co"># Ver os modelos disponíveis</span></span>
<span id="cb3-2"><a href=""></a><span class="ex">dorado</span> download <span class="at">--list</span></span>
<span id="cb3-3"><a href=""></a></span>
<span id="cb3-4"><a href=""></a><span class="co"># Baixar o desejado na pasta desejada</span></span>
<span id="cb3-5"><a href=""></a><span class="ex">dorado</span> download <span class="at">--model</span> {modelo} <span class="at">--models-directory</span> {PATH}</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="modelos-dorado-1" class="slide level2">
<h2>Modelos <code>dorado</code></h2>
<ul>
<li><p><code>FAST</code>: Mais rápido e com menor acurácia;</p></li>
<li><p><code>HAC</code>: Alta acurácia;</p></li>
<li><p><code>SUP</code>: Super acurácia (maior requisito computacional);</p></li>
</ul>
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Para escolher seu modelo.</strong></p>
</div>
<div class="callout-content">
<p>Modelo <code>SUP</code> é uma vantagem em situações de construção de genomas de referência;</p>
<p>Para demais situações o <code>HAC</code> já é suficiente;</p>
</div>
</div>
</div>
</section>
<section id="dorado-basecaller" class="slide level2">
<h2><code>dorado</code> <code>basecaller</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb4-1"><a href=""></a><span class="co"># chamada HAC sem modificações</span></span>
<span id="cb4-2"><a href=""></a><span class="ex">dorado</span> basecaller hac {/path/to/reads.pod5} <span class="op">&gt;</span> calls.bam</span>
<span id="cb4-3"><a href=""></a></span>
<span id="cb4-4"><a href=""></a><span class="co">#chamada HAC com modificações 6mA e 5mC e 5hmC em CpG</span></span>
<span id="cb4-5"><a href=""></a><span class="ex">dorado</span> basecaller hac,6mA,5mCG_5hmCG {/path/to/reads.pod5} <span class="op">&gt;</span> calls.bam</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="dorado-basecaller-1" class="slide level2">
<h2><code>dorado</code> <code>basecaller</code></h2>
<ul>
<li><p>O padrão é salvar como <code>BAM</code>;</p>
<ul>
<li>Isso vai permitir salvar as modificações de base;</li>
</ul></li>
<li><p>A performance ideal requer GPU;</p>
<ul>
<li>Usar a tag <code>--device "cuda:all"</code>;</li>
</ul></li>
<li><p>Para gerar <code>fastq</code> usar <code>–emit-fastq</code>;</p></li>
<li><p>Para novos modelos é necessário utilizar <code>–emit-moves</code>;</p></li>
<li><p>É possível chamar e alinhar ao genoma de referência usando <code>--reference</code>;</p></li>
</ul>
</section>
<section id="dorado-basecaller-2" class="slide level2">
<h2><code>dorado</code> <code>basecaller</code></h2>
<div class="callout callout-note callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Tempo de processamento</strong></p>
</div>
<div class="callout-content">
<p>~ 12 horas de chamada de base + alinhamento utilizando GPU RTX 4060;</p>
<p><code>{model}</code> precisa ser o caminho completo;</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb5-1"><a href=""></a><span class="ex">dorado</span> basecaller <span class="at">--recursive</span> <span class="at">--trim</span> <span class="st">"all"</span> <span class="at">--modified-bases</span> 5mCG_5hmCG 6mA <span class="at">--reference</span> {fasta} {modelo} pod5/ <span class="at">--device</span> <span class="st">"cuda:all"</span> <span class="op">&gt;</span> calls.bam</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</div>
</div>
</div>
</section>
<section id="qc" class="slide level2">
<h2>QC</h2>
<ul>
<li><p><code>FASTQC</code> tambem funciona;</p></li>
<li><p><code>NanoPack2</code> é a atual melhor opção;</p>
<ul>
<li><p><code>NanoPlot</code> para visualização das estatísticas;</p></li>
<li><p><code>Chopper</code> para trimagem (FASTQ);</p></li>
</ul></li>
</ul>
</section>
<section id="nanoplot" class="slide level2">
<h2>NanoPlot</h2>
<ul>
<li><p>FASTQ ou BAM;</p></li>
<li><p>Qualidade (<code>Q</code>);</p></li>
<li><p>Tamanho (<code>L</code>);</p></li>
<li><p>N50;</p></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://hoytpr.github.io/bioinformatics-semester/materials/genomics-assembly-reporting/"><img data-src="https://hoytpr.github.io/bioinformatics-semester/fig/N50.png" class="quarto-figure quarto-figure-center" width="615"></a></p>
</figure>
</div>
</section>
<section id="nanoplot-1" class="slide level2">
<h2>NanoPlot</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb6-1"><a href=""></a><span class="ex">NanoPlot</span> <span class="at">-t</span> {threads} {--fastq/--bam} {files} <span class="at">-o</span> {OUTPUT_DIR_NAME}</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>

<img data-src="https://github.com/wdecoster/NanoPlot/raw/master/examples/scaled_Log_Downsampled_LengthvsQualityScatterPlot_kde.png" width="850" class="r-stretch quarto-figure-center"><p class="caption">Exemplo de gráfico</p></section>
<section id="chopper" class="slide level2">
<h2>Chopper</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb7-1"><a href=""></a><span class="ex">chopper</span> <span class="at">-q</span> 20 <span class="at">-l</span> 1000 reads.fastq <span class="op">&gt;</span> trimmed_reads.fastq </span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="samtools" class="slide level2">
<h2>Samtools</h2>
<ul>
<li>Filtrar o bam usando samtools (v1.17+):</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb8-1"><a href=""></a><span class="ex">samtools</span> view <span class="dt">\</span></span>
<span id="cb8-2"><a href=""></a>    <span class="at">--threads</span> 16 <span class="dt">\</span></span>
<span id="cb8-3"><a href=""></a>    <span class="at">--bam</span> <span class="at">--with-header</span> <span class="dt">\</span></span>
<span id="cb8-4"><a href=""></a>    <span class="at">--min-qlen</span> 500 <span class="dt">\</span></span>
<span id="cb8-5"><a href=""></a>    <span class="at">--min-MQ</span> 10 <span class="dt">\</span></span>
<span id="cb8-6"><a href=""></a>    <span class="at">--exclude-flags</span> 260 <span class="dt">\</span></span>
<span id="cb8-7"><a href=""></a>    <span class="at">--expr</span> <span class="st">"avg(qual) &gt;= 20 &amp;&amp; qlen &gt;= 1000"</span> <span class="dt">\</span></span>
<span id="cb8-8"><a href=""></a>    {infile} <span class="op">&gt;</span> {outfile}</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="variantes" class="slide level2">
<h2>Variantes</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genomic-variation"><img data-src="https://www.genome.gov/sites/default/files/inline-images/Genomic%20variation_INDELS%20DEL%20SNV%20TANDEM.png" class="quarto-figure quarto-figure-center" width="1000" alt="Variantes."></a></p>
</figure>
</div>
<figcaption>Variantes.</figcaption>
</figure>
</div>
</section>
<section id="diferentes-softwares" class="slide level2">
<h2>Diferentes softwares</h2>
<ul>
<li><p>GATK;</p></li>
<li><p>DeepVariant;</p></li>
<li><p>Freebayes;</p></li>
<li><p>clair3;</p></li>
<li><p>samtools + Picard;</p></li>
<li><p>Entre outros…</p></li>
</ul>
</section>
<section id="freebayes" class="slide level2">
<h2><code>freebayes</code></h2>
<ul>
<li><p>Modelagem bayesiana;</p></li>
<li><p>Monta pequenos haplótipos;</p></li>
<li><p>Criado para short-reads;</p></li>
<li><p>Propenso a erros do tipo 1 (Falsos positivos);</p></li>
<li><p>Pode ser usado para uma ou várias amostras;</p></li>
</ul>
</section>
<section id="comando-freebayes" class="slide level2">
<h2>Comando <code>freebayes</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb9-1"><a href=""></a><span class="co"># Uma amostra</span></span>
<span id="cb9-2"><a href=""></a><span class="ex">freebayes</span> <span class="at">-f</span> {REF} <span class="at">-t</span> {REGIONS.BED} {amostra.bam} <span class="op">&gt;</span> var.vcf</span>
<span id="cb9-3"><a href=""></a></span>
<span id="cb9-4"><a href=""></a><span class="co"># Multisample focando em uma região</span></span>
<span id="cb9-5"><a href=""></a><span class="fu">ls</span> <span class="pp">*</span>.bam <span class="op">&gt;</span> BAM_LIST.txt</span>
<span id="cb9-6"><a href=""></a><span class="ex">freebayes</span> <span class="at">-f</span> {REF} <span class="at">-L</span> {BAM_LIST.txt} <span class="at">-t</span> {REGIONS.BED} <span class="op">&gt;</span> var.vcf</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="freebayes-não-é-paralelizavel" class="slide level2">
<h2><code>freebayes</code> não é paralelizavel</h2>
<div class="fragment">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="freebayes_sw.png" class="quarto-figure quarto-figure-center" width="1000"></p>
</figure>
</div>
</div>
</section>
<section id="freebayes-não-é-paralelizavel-1" class="slide level2">
<h2><code>freebayes</code> não é paralelizavel</h2>
<p>Parametros já testados:</p>
<ul>
<li><p>Windows = 1000 bp;</p></li>
<li><p>Overlap = 50 bp;</p></li>
<li><p>Step size = 950 bp (1000 - 50);</p></li>
</ul>
</section>
<section id="combinando-todos-os-arquivos" class="slide level2">
<h2>Combinando todos os arquivos</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb10-1"><a href=""></a><span class="cf">for</span> file <span class="kw">in</span> path/to/data/<span class="pp">*</span>.vcf<span class="kw">;</span> <span class="cf">do</span></span>
<span id="cb10-2"><a href=""></a>    <span class="ex">bgzip</span> <span class="at">-c</span> <span class="st">"</span><span class="va">$file</span><span class="st">"</span> <span class="op">&gt;</span> <span class="st">"</span><span class="va">$file</span><span class="st">.gz"</span></span>
<span id="cb10-3"><a href=""></a>    <span class="ex">tabix</span> <span class="at">-p</span> vcf <span class="st">"</span><span class="va">$file</span><span class="st">.gz"</span></span>
<span id="cb10-4"><a href=""></a><span class="cf">done</span></span>
<span id="cb10-5"><a href=""></a></span>
<span id="cb10-6"><a href=""></a></span>
<span id="cb10-7"><a href=""></a><span class="ex">bcftools</span> concat <span class="at">--threads</span> 14 <span class="at">--allow-overlaps</span> <span class="at">--remove-duplicates</span> <span class="at">-o</span> ../merged.vcf.gz <span class="at">-Oz</span> <span class="va">$(</span><span class="fu">ls</span> <span class="pp">*</span>.vcf.gz<span class="va">)</span></span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="filtrando-arquivos-vcf" class="slide level2">
<h2>Filtrando arquivos VCF</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb11-1"><a href=""></a><span class="co"># remove variantes de qualidade menor que 1</span></span>
<span id="cb11-2"><a href=""></a><span class="ex">bcftools</span> view <span class="at">--exclude</span> <span class="st">'QUAL&lt;1'</span> variantes.vcf <span class="op">&gt;</span> variantes_filtradas.vcf</span>
<span id="cb11-3"><a href=""></a></span>
<span id="cb11-4"><a href=""></a><span class="co"># remove alelos alternativos que não apareceram</span></span>
<span id="cb11-5"><a href=""></a><span class="ex">bcftools</span> view <span class="at">--trim-alt-alleles</span> variantes_filtradas.vcf <span class="op">&gt;</span> variantes_filtradas_trim.vcf</span>
<span id="cb11-6"><a href=""></a></span>
<span id="cb11-7"><a href=""></a><span class="co"># remove variantes que não tenham nenhum alelo alternativo nas amostras</span></span>
<span id="cb11-8"><a href=""></a><span class="ex">bcftools</span> view <span class="at">--min-ac</span> 1 variantes_filtradas_trim.vcf <span class="op">&gt;</span> variantes_filtradas_trim_minac.vcf</span>
<span id="cb11-9"><a href=""></a></span>
<span id="cb11-10"><a href=""></a><span class="co"># normaliza o vcf com base no genoma de referência</span></span>
<span id="cb11-11"><a href=""></a><span class="ex">bcftools</span> norm <span class="at">-f</span> hg38.fa variantes_filtradas_trim_minac.vcf <span class="op">&gt;</span> variantes_filtradas_trim_minac_norm.vcf</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="clair3" class="slide level2">
<h2><code>clair3</code></h2>
<ul>
<li><p>Desenhada para dados de diferentes plataformas;</p>
<ul>
<li>ONT, PacBio, Illumina;</li>
</ul></li>
<li><p>Usa deep learning;</p></li>
<li><p>Modos <code>pileup</code> e <code>full aligment</code> (<code>merge</code> combinação);</p></li>
<li><p>Roda uma amostra por vez;</p>
<ul>
<li><code>--gvcf</code>;</li>
</ul></li>
</ul>
</section>
<section id="comando-clair3" class="slide level2">
<h2>Comando <code>clair3</code></h2>
<ul>
<li>Usando <code>bioconda</code>:</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb12-1"><a href=""></a><span class="va">MODEL_NAME</span><span class="op">=</span><span class="st">"[YOUR_MODEL_NAME]"</span>         <span class="co"># e.g. r1041_e82_400bps_sup_v500</span></span>
<span id="cb12-2"><a href=""></a><span class="va">THREADS</span><span class="op">=</span>4</span>
<span id="cb12-3"><a href=""></a><span class="va">OUTPUT_DIR</span><span class="op">=</span><span class="st">"ondesalvar"</span></span>
<span id="cb12-4"><a href=""></a><span class="ex">run_clair3.sh</span> <span class="dt">\</span></span>
<span id="cb12-5"><a href=""></a>  <span class="at">--bam_fn</span><span class="op">=</span>input.bam <span class="dt">\ </span>                <span class="co">## change your bam file name here</span></span>
<span id="cb12-6"><a href=""></a>  <span class="ex">--ref_fn=ref.fa</span> <span class="dt">\ </span>                   <span class="co">## change your reference file name here</span></span>
<span id="cb12-7"><a href=""></a>  <span class="ex">--threads=</span><span class="va">${THREADS}</span> <span class="dt">\ </span>              <span class="co">## maximum threads to be used</span></span>
<span id="cb12-8"><a href=""></a>  <span class="ex">--platform=</span><span class="st">"ont"</span> <span class="dt">\ </span>                  <span class="co">## options: {ont,hifi,ilmn}</span></span>
<span id="cb12-9"><a href=""></a>  <span class="ex">--model_path=</span><span class="st">"</span><span class="va">${CONDA_PREFIX}</span><span class="st">/bin/models/</span><span class="va">${MODEL_NAME}</span><span class="st">"</span> <span class="dt">\ </span></span>
<span id="cb12-10"><a href=""></a>  <span class="ex">--output=</span><span class="va">${OUTPUT_DIR}</span> <span class="dt">\ </span>             <span class="co">## output path prefix </span></span>
<span id="cb12-11"><a href=""></a>  <span class="ex">--sample_name=</span><span class="st">"nomedaamostra"</span> <span class="dt">\</span></span>
<span id="cb12-12"><a href=""></a>  <span class="at">--gvcf</span> <span class="dt">\</span></span>
<span id="cb12-13"><a href=""></a>  <span class="at">--bed_fn</span><span class="op">=</span> <span class="st">"genes_disciplina.bed"</span></span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="multi-amostras" class="slide level2">
<h2>Multi amostras</h2>
<ul>
<li><p><code>GLnexus</code>;</p></li>
<li><p>Modelo para usar [<a href="http://www.bio8.cs.hku.hk/clair3_trio/config/clair3.yml">link</a>];</p></li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb13-1"><a href=""></a><span class="co"># rodar o glnexus para juntar as amostras</span></span>
<span id="cb13-2"><a href=""></a><span class="ex">glnexus_cli</span> <span class="at">--config</span> {clair3.yml} <span class="at">--threads</span> 12 [1.gvcf, 2.gvcf, 3.gvcf...] <span class="op">&gt;</span> all_samples.bcf</span>
<span id="cb13-3"><a href=""></a></span>
<span id="cb13-4"><a href=""></a><span class="co"># transformar em vcf</span></span>
<span id="cb13-5"><a href=""></a><span class="ex">bcftools</span> view <span class="at">-Ov</span> <span class="at">-o</span> all_samples.vcf all_samples.bcf</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="variantes-estruturais-sv" class="slide level2">
<h2>Variantes estruturais (SV)</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.genome.gov/about-genomics/educational-resources/fact-sheets/human-genomic-variation"><img data-src="https://www.genome.gov/sites/default/files/inline-images/Genomic%20variation_Structral%20variants.png" class="quarto-figure quarto-figure-center" width="1000" alt="Variantes estruturais."></a></p>
</figure>
</div>
<figcaption>Variantes estruturais.</figcaption>
</figure>
</div>
</section>
<section id="programas" class="slide level2">
<h2>Programas</h2>
<ul>
<li>Sniffles2;</li>
</ul>
<p><br></p>
<ul>
<li>SVIM;</li>
</ul>
<p><br></p>
<ul>
<li><a href="https://github.com/tjiangHIT/cuteSV">cuteSV</a>;</li>
</ul>
</section>
<section id="sniffles2" class="slide level2">
<h2>Sniffles2</h2>
<ul>
<li>Permite análise de TRIOS (Pais e filhos), populações (multiplas amostras) e mosaico;</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb14-1"><a href=""></a><span class="co"># Padrão</span></span>
<span id="cb14-2"><a href=""></a><span class="ex">sniffles</span> <span class="at">--input</span> {amostra.bam} <span class="at">--snf</span> {amostra.snf} <span class="at">--vcf</span> {amostra.vcf}</span>
<span id="cb14-3"><a href=""></a></span>
<span id="cb14-4"><a href=""></a><span class="co">#Multisample</span></span>
<span id="cb14-5"><a href=""></a><span class="ex">sniffles</span> <span class="at">--input</span> {amostra1.snf} {amostra2.snf} ... <span class="at">--vcf</span> {todasamostra.vcf}</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="faseamento" class="slide level2">
<h2>Faseamento</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://wp.unil.ch/gtf/applications/"><img data-src="https://wp.unil.ch/gtf/files/2020/05/EtNR1T.png" class="quarto-figure quarto-figure-center" alt="Vantagens no faseamento"></a></p>
</figure>
</div>
<figcaption>Vantagens no faseamento</figcaption>
</figure>
</div>
</section>
<section id="programas-1" class="slide level2">
<h2>Programas</h2>
<ul>
<li><a href="https://whatshap.readthedocs.io/en/latest/">WhatsHap</a>;</li>
</ul>
<p><br></p>
<ul>
<li><a href="https://github.com/twolinin/longphase">Longphase</a>;</li>
</ul>
<p><br></p>
<ul>
<li><a href="https://github.com/vibansal/HapCUT2">HapCUT2</a>;</li>
</ul>
</section>
<section id="whatshap" class="slide level2">
<h2>WhatsHap</h2>
<ul>
<li><p>Permite trabalhar com &gt;1 amostra no vcf;</p></li>
<li><p>Não lida com variantes multialelicas;</p></li>
<li><p>Não lida com SVs;</p></li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb15-1"><a href=""></a><span class="co"># Phase</span></span>
<span id="cb15-2"><a href=""></a><span class="ex">whatshap</span> phase <span class="at">--indels</span> <span class="at">-o</span> phased.vcf <span class="at">--reference</span><span class="op">=</span>reference.fasta input.vcf input.bam</span>
<span id="cb15-3"><a href=""></a></span>
<span id="cb15-4"><a href=""></a><span class="co">#Haplotag</span></span>
<span id="cb15-5"><a href=""></a><span class="ex">whatshap</span> haplotag <span class="at">-o</span> haplotagged.bam <span class="at">--reference</span> reference.fasta phased.vcf.gz alignments.bam</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="longphase" class="slide level2">
<h2>Longphase</h2>
<ul>
<li><p>Permite apenas 1 amostra por vcf;</p></li>
<li><p>Suporta com SVs;</p></li>
<li><p>É capaz de utilizar a metilação para a fase;</p></li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb16-1"><a href=""></a><span class="co"># Phase. --sv-file passa o vcf de SVs</span></span>
<span id="cb16-2"><a href=""></a><span class="ex">longphase</span> phase <span class="at">-s</span> SNP.vcf <span class="at">-b</span> alignment.bam <span class="at">-r</span> reference.fasta <span class="at">-t</span> 8  <span class="at">-o</span> phased_prefix <span class="at">--indels</span> <span class="at">--ont</span></span>
<span id="cb16-3"><a href=""></a></span>
<span id="cb16-4"><a href=""></a><span class="co"># Haplotag </span></span>
<span id="cb16-5"><a href=""></a><span class="ex">longphase</span> haplotag <span class="at">-r</span> reference.fasta <span class="at">-s</span> phased_snp.vcf <span class="at">--sv-file</span> phased_sv.vcf <span class="at">-b</span> alignment.bam <span class="at">-t</span> 8 <span class="at">-o</span> tagged_bam_prefix</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="montagem-de-genomas" class="slide level2">
<h2>Montagem de genomas</h2>
<ul>
<li><p><strong>Shasta</strong>: algoritmo de grafo string, extremamente rápido;</p></li>
<li><p><strong>Verkko</strong>: híbrido ONT+HiFi, qualidade superior;</p></li>
<li><p><strong>Hifiasm</strong>: originalmente para HiFi, adaptado para ONT;</p></li>
<li><p><strong>PECAT:</strong> versátil, realiza correção e polimento;</p></li>
</ul>
</section>
<section id="programas-2" class="slide level2">
<h2>Programas</h2>
<ul>
<li><a href="https://github.com/lemene/PECAT">PECAT</a>;</li>
</ul>
<p><br></p>
<ul>
<li><a href="https://github.com/chhylp123/hifiasm">Hifiasm</a>;</li>
</ul>
</section>
<section id="pecat" class="slide level2">
<h2>PECAT</h2>
<ul>
<li><p>Coleção de ferramentas;</p></li>
<li><p>minimap2;</p></li>
<li><p>clair3;</p></li>
<li><p>medaka;</p></li>
</ul>
</section>
<section id="pecat-1" class="slide level2">
<h2>PECAT</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.nature.com/articles/s41467-024-47349-7/figures/4"><img data-src="images/pecat.webp" class="quarto-figure quarto-figure-center" width="1350"></a></p>
</figure>
</div>
</section>
<section id="pecat-2" class="slide level2">
<h2>PECAT</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb17-1"><a href=""></a><span class="co"># Ver opções</span></span>
<span id="cb17-2"><a href=""></a><span class="ex">pecat.pl</span></span>
<span id="cb17-3"><a href=""></a></span>
<span id="cb17-4"><a href=""></a><span class="co"># Cria um arquivo texto com todos os parametros</span></span>
<span id="cb17-5"><a href=""></a><span class="ex">pecat.pl</span> config {cfg_name}</span>
<span id="cb17-6"><a href=""></a></span>
<span id="cb17-7"><a href=""></a><span class="co"># Roda</span></span>
<span id="cb17-8"><a href=""></a><span class="ex">pecat.pl</span> unzip {cfg_name}</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="hifiasm" class="slide level2">
<h2>Hifiasm</h2>
<ul>
<li><p>Desenhado para Hifi reads(PacBio), adaptado para ONT;</p></li>
<li><p>Gerar grafos;</p></li>
<li><p>Funciona para diploides;</p></li>
<li><p>Rápido;</p></li>
</ul>
</section>
<section id="alinhando-os-ctgs-ao-genoma-de-referencia" class="slide level2">
<h2>Alinhando os ctgs ao genoma de referencia</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb18-1"><a href=""></a><span class="co">#Juntar todos os ctgs num unico arquivo</span></span>
<span id="cb18-2"><a href=""></a></span>
<span id="cb18-3"><a href=""></a><span class="co"># minimap asm20</span></span>
<span id="cb18-4"><a href=""></a><span class="ex">minimap2</span> <span class="at">-t</span> 10 <span class="at">-ax</span> asm20 <span class="at">-B</span> 2 <span class="at">-E</span> 3,1 <span class="at">-O</span> 6,100 <span class="at">--cs</span> /path/to/genomes/reference.fasta OCA2_ctgs.fasta <span class="op">&gt;</span> OCA2_ctgs.sam</span>
<span id="cb18-5"><a href=""></a></span>
<span id="cb18-6"><a href=""></a><span class="co"># Transformar em bam</span></span>
<span id="cb18-7"><a href=""></a><span class="ex">samtools</span> sort OCA2_ctgs.sam <span class="op">&gt;</span> OCA2_ctgs.bam</span>
<span id="cb18-8"><a href=""></a><span class="ex">samtools</span> index OCA2_ctgs.bam</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="modificação-de-bases" class="slide level2">
<h2>Modificação de bases</h2>
</section>
<section id="modbam" class="slide level2">
<h2>Modbam</h2>
<p>- O produto de chamada de base é chamado de modbam. Embora a extensão seja <code>.bam</code>;</p>
<p>- São adicionadas duas tags no bam contendo as informações de modificação: <code>MM</code> e <code>ML</code>;</p>
<p>- <code>IGV</code>;</p>
</section>
<section id="modkit" class="slide level2">
<h2>Modkit</h2>
<ul>
<li>Kit de ferramentas desenvolvida pela Oxford Nanopore Technologies;</li>
</ul>
<!-- -->
<ul>
<li><p>Pipeline completo: desde BAM files até visualizações finais;</p></li>
<li><p>Interface command-line intuitiva e <a href="https://nanoporetech.github.io/modkit/">bem documentada</a>;</p></li>
</ul>
</section>
<section id="modkit-1" class="slide level2">
<h2>Modkit</h2>
<ul>
<li><p><code>pileup</code>: Agregação de probabilidades de modificação por posição genômica;</p></li>
<li><p><code>DMR</code>: Calculo de regiões diferencialmente metiladas;</p></li>
<li><p><code>Entropy</code>: medida de heterogeneidade de modificação em população celular;</p></li>
</ul>
</section>
<section id="modkit---pileup" class="slide level2">
<h2>Modkit - pileup</h2>
<ul>
<li><p>Gera um bedmethyl como resultado;</p></li>
<li><p>Pode ser para genoma completo o posições específicas;</p></li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb19-1"><a href=""></a><span class="co"># Pileup padrão</span></span>
<span id="cb19-2"><a href=""></a><span class="ex">modkit</span> pileup path/to/reads.bam output/path/pileup.bed <span class="at">--cpg</span> <span class="at">--ref</span> path/to/reference.fasta</span>
<span id="cb19-3"><a href=""></a></span>
<span id="cb19-4"><a href=""></a><span class="co"># Pileup por região. Ex CpG</span></span>
<span id="cb19-5"><a href=""></a><span class="ex">modkit</span> pileup path/to/reads.bam output/path/pileup.bed <span class="dt">\</span></span>
<span id="cb19-6"><a href=""></a>  <span class="at">--cpg</span> <span class="dt">\</span></span>
<span id="cb19-7"><a href=""></a>  <span class="at">--ref</span> path/to/reference.fasta <span class="dt">\</span></span>
<span id="cb19-8"><a href=""></a>  <span class="at">--include-bed</span> path/to/my_cpgs.bed</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="modkit---pileup-por-hp" class="slide level2">
<h2>Modkit - pileup por HP</h2>
<ul>
<li>É possível analisar por haplótipo:</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb20-1"><a href=""></a><span class="ex">modkit</span> pileup path/to/reads.bam output/directory/ <span class="at">--cpg</span> <span class="at">--ref</span> <span class="op">&lt;</span>reference.fasta<span class="op">&gt;</span> --partition-tag HP <span class="at">--prefix</span> haplotyped</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="modkit---dmr" class="slide level2">
<h2>Modkit - DMR</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb21-1"><a href=""></a><span class="co"># Gerar os pileups para as amostras individualmente</span></span>
<span id="cb21-2"><a href=""></a></span>
<span id="cb21-3"><a href=""></a><span class="co"># bgzip e tabix para os beds</span></span>
<span id="cb21-4"><a href=""></a><span class="ex">bgzip</span> <span class="at">-k</span> <span class="va">${pileup}</span></span>
<span id="cb21-5"><a href=""></a><span class="ex">tabix</span> <span class="at">-p</span> bed <span class="va">${pileup}</span>.gz</span>
<span id="cb21-6"><a href=""></a></span>
<span id="cb21-7"><a href=""></a><span class="co"># dmr pair para pares</span></span>
<span id="cb21-8"><a href=""></a><span class="ex">modkit</span> dmr pair <span class="at">-a</span> {amostra1.bed.gz} <span class="at">-b</span> {amostra2.bed.gz} <span class="at">-o</span> {output_dir} <span class="at">-r</span> {regioes.bed} <span class="at">--ref</span> {genoma.fa} <span class="at">--base</span> C <span class="at">--threads</span> 10</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="modkit---entropy" class="slide level2">
<h2>Modkit - entropy</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb22-1"><a href=""></a><span class="ex">modkit</span> entropy <span class="at">--in-bam</span> <span class="va">${mod_bam}</span> <span class="dt">\</span></span>
<span id="cb22-2"><a href=""></a> <span class="at">-o</span> <span class="va">${output_directory}</span> <span class="dt">\</span></span>
<span id="cb22-3"><a href=""></a> <span class="at">--regions</span> <span class="va">${regions_bed_file}</span> <span class="dt">\ </span> <span class="co"># BED3 or BED4 file of regions</span></span>
<span id="cb22-4"><a href=""></a> <span class="ex">--cpg</span> <span class="dt">\ </span> <span class="co"># specify CpG dinucleotides and combine strands</span></span>
<span id="cb22-5"><a href=""></a> <span class="ex">--ref</span> <span class="va">${ref}</span> <span class="dt">\</span></span>
<span id="cb22-6"><a href=""></a> <span class="at">--threads</span> 10</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="modkit---extras" class="slide level2">
<h2>Modkit - extras</h2>
<ul>
<li><p><a href="https://nanoporetech.github.io/modkit/intro_localize.html">Localize</a></p></li>
<li><p><a href="https://nanoporetech.github.io/modkit/intro_open_chromatin.html">Open Chromatin</a></p></li>
</ul>

</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<div class="footer footer-default">
<p>marcel.ferreira@unesp.br</p>
</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="bioinfo-nanopore_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/reveal-pointer/pointer.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="bioinfo-nanopore_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"8\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true},
'pointer': {"key":"q","color":"#ff0800","pointerSize":16,"alwaysVisible":false},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'slow',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1500,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, RevealPointer, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    

    <script>

      // htmlwidgets need to know to resize themselves when slides are shown/hidden.

      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current

      // slide changes (different for each slide format).

      (function () {

        // dispatch for htmlwidgets

        function fireSlideEnter() {

          const event = window.document.createEvent("Event");

          event.initEvent("slideenter", true, true);

          window.document.dispatchEvent(event);

        }

    

        function fireSlideChanged(previousSlide, currentSlide) {

          fireSlideEnter();

    

          // dispatch for shiny

          if (window.jQuery) {

            if (previousSlide) {

              window.jQuery(previousSlide).trigger("hidden");

            }

            if (currentSlide) {

              window.jQuery(currentSlide).trigger("shown");

            }

          }

        }

    

        // hookup for slidy

        if (window.w3c_slidy) {

          window.w3c_slidy.add_observer(function (slide_num) {

            // slide_num starts at position 1

            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);

          });

        }

    

      })();

    </script>

    

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copiada");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copiada");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp('/' + window.location.host + '/');
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>