<!DOCTYPE html>
<html lang="pt-BR"><head>
<script src="nanopore_files/libs/clipboard/clipboard.min.js"></script>
<script src="nanopore_files/libs/quarto-html/tabby.min.js"></script>
<script src="nanopore_files/libs/quarto-html/popper.min.js"></script>
<script src="nanopore_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="nanopore_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="nanopore_files/libs/quarto-html/light-border.css" rel="stylesheet">
<link href="nanopore_files/libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.6.39">

  <meta name="author" content="Marcel Ferreira">
  <meta name="dcterms.date" content="2025-08-22">
  <title>Sequenciamento de Terceira Geração com Oxford Nanopore</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="nanopore_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="nanopore_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for syntax highlighting */
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; font-weight: bold; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="nanopore_files/libs/revealjs/dist/theme/quarto-423d414d36ce366a9d05f36d06304417.css">
  <link href="nanopore_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Sequenciamento de Terceira Geração com Oxford Nanopore</h1>
  <p class="subtitle">Princípios e Aplicações em Bioinformática</p>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
Marcel Ferreira 
</div>
</div>
</div>

  <p class="date">2025-08-22</p>
</section>
<section>
<section id="sobre-mim" class="title-slide slide level1 center">
<h1>Sobre mim</h1>

</section>
<section id="muito-prazer.." class="slide level2">
<h2>Muito prazer..</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/marshall.jpg" width="420"></p>
<figcaption>Olar!</figcaption>
</figure>
</div>
</div><div class="column" style="font-size: 80%;">
<ul>
<li>Marcel;</li>
</ul>
<p><br></p>
<ul>
<li>Físico médico;</li>
</ul>
<p><br></p>
<ul>
<li>Me e PhD em Biotecnologia;</li>
</ul>
<p><br></p>
<ul>
<li>Desde 2023, pós-doutorado em genética;</li>
</ul>
</div></div>
</section>
<section id="interesses" class="slide level2">
<h2>Interesses</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/marshall.jpg" width="420"></p>
<figcaption>Olar!</figcaption>
</figure>
</div>
</div><div class="column" style="font-size: 80%;">
<ul>
<li><p>Métodos alternativos ao uso de animais;</p></li>
<li><p>Identificão humana via DNA;</p></li>
<li><p>Biologia do tecido ósseo;</p></li>
<li><p>Biomateriais;</p></li>
<li><p>Regeneração tecidual;</p></li>
<li><p>Ciência aberta;</p></li>
</ul>
<p><br></p>
<div id="red-highlight" style="color: red; font-size: 140%; text-align: center; font-weight: bold;">
<p>Bioinformática</p>
</div>
</div></div>
</section>
<section id="projeto" class="slide level2">
<h2>Projeto</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://isbet.org.br/diversidade-no-estagio-e-jovem-aprendiz/"><img data-src="https://isbet.org.br/wp-content/uploads/2024/08/grupo-de-jovens-multirraciais-sorrindo-para-a-camera-conceito-de-amizade-e-diversidade_442523-605.webp" class="quarto-figure quarto-figure-center" width="780" alt="Diversidade: como podemos utilizar DNA para predizer o fenótipo?"></a></p>
</figure>
</div>
<figcaption>Diversidade: como podemos utilizar DNA para predizer o fenótipo?</figcaption>
</figure>
</div>
</section>
<section id="não-sou-da-nanopore" class="slide level2">
<h2>Não sou da nanopore</h2>
<div style="text-align: center;">
<iframe src="https://giphy.com/embed/zFa3vjySsETPa" width="414" height="480" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/zFa3vjySsETPa">via GIPHY</a>
</p>
</div>
</section></section>
<section>
<section id="sobre-vocês" class="title-slide slide level1 center">
<h1>Sobre vocês</h1>

</section>
<section id="sobre-vocês-1" class="slide level2">
<h2>Sobre vocês</h2>
<div style="text-align: center;">
<iframe src="https://giphy.com/embed/5wWf7H89PisM6An8UAU" width="960" height="558" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/editingandlayout-the-office-michael-scott-5wWf7H89PisM6An8UAU">via GIPHY</a>
</p>
</div>
</section></section>
<section>
<section id="sobre-essa-disciplina" class="title-slide slide level1 center">
<h1>Sobre essa disciplina</h1>

</section>
<section id="primeira-edição-de-muitas" class="slide level2">
<h2>Primeira edição (de muitas…🙏)</h2>
<ul>
<li>Desde maio estou como colaborador no programa de Genética do IBB;</li>
</ul>
<p><br></p>
<ul>
<li>Infelizmente não terá pratica de bancada;</li>
</ul>
<p><br></p>
<ul>
<li>Vamos aprofundar sobre aspectos de bioinformática das análises;</li>
</ul>
<p><br></p>
<ul>
<li>Vamos focar em <strong><em>genômica</em></strong> e <strong><em>epigenômica</em></strong>;</li>
</ul>
</section>
<section id="desafios" class="slide level2">
<h2>Desafios 🚵</h2>
<ul>
<li><p>Bancada..</p></li>
<li><p>Requisitos computacionais…</p></li>
<li><p>Tempo…</p></li>
<li><p>Programas…</p></li>
</ul>
</section>
<section id="material-online" class="slide level2">
<h2>Material online</h2>
<ul>
<li><a href="https://marceelrf.quarto.pub/nanopore_genetica">Esses slides</a>;</li>
</ul>
<p><br></p>
<ul>
<li><a href="https://marceelrf-material.netlify.app/nanopore.html">Livro de códigos</a>;</li>
</ul>
</section>
<section id="fruto-do-meu-projeto" class="slide level2">
<h2>Fruto do meu projeto</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://workshopbioinfoforense.netlify.app/"><img data-src="images/Slide2.PNG" class="quarto-figure quarto-figure-center" width="462" alt="Workshop realizado em 2023 e 2024."></a></p>
</figure>
</div>
<figcaption>Workshop realizado em 2023 e 2024.</figcaption>
</figure>
</div>
</section>
<section id="método-do-ano-de-2022" class="slide level2">
<h2>Método do ano de 2022</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.nature.com/articles/s41592-022-01730-w"><img data-src="methodoftheyear.png" class="quarto-figure quarto-figure-center" width="516" alt="Publicado na Nature:"></a></p>
</figure>
</div>
<figcaption>Publicado na Nature:</figcaption>
</figure>
</div>
</section>
<section id="passar-os-atalhos" class="slide level2">
<h2>Passar os atalhos</h2>
<div class="columns">
<div class="column" style="width:50%;">
<iframe src="https://giphy.com/embed/1n833bZxdzKzaErLe9" width="480" height="269" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/nba-expression-derrick-white-1n833bZxdzKzaErLe9">via GIPHY</a>
</p>
</div><div class="column" style="width:50%;">
<iframe src="https://giphy.com/embed/Cas0va26zqBvTO5vY9" width="480" height="480" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/pudgypenguins-rage-enough-ragequit-Cas0va26zqBvTO5vY9">via GIPHY</a>
</p>
</div></div>
</section>
<section id="bioinformática" class="slide level2">
<h2>Bioinformática</h2>
<ul>
<li>NIH: “<em>Bioinformatics, as related to genetics and genomics, is a scientific subdiscipline that involves using computer technology to collect, store, analyze and disseminate biological data and information, such as DNA and amino acid sequences or annotations about those sequences.</em>”</li>
</ul>

<aside><div>
<p>NIH: National Institute of Health</p>
</div></aside></section>
<section id="bioinformática-1" class="slide level2">
<h2>Bioinformática</h2>
<ul>
<li>NIH: “<em>Bioinformatics, as related to <strong>genetics</strong> and <strong>genomics</strong>, is a scientific subdiscipline that involves using computer technology to <strong>collect</strong>, <strong>store</strong>, <strong>analyze</strong> and <strong>disseminate</strong> biological data and information, such as <strong>DNA</strong> and <strong>amino acid sequences</strong> or annotations about those sequences.</em>”</li>
</ul>

<aside><div>
<p>NIH: National Institute of Health</p>
</div></aside></section>
<section id="bioinformática-2" class="slide level2">
<h2>Bioinformática</h2>
<ul>
<li>Não vamos focar nos algoritmos;</li>
</ul>
<p><br></p>
<ul>
<li>Mas sim nos dados produzidos e como analisa-los;</li>
</ul>
</section>
<section id="cronograma" class="slide level2">
<h2>Cronograma</h2>
<div class="columns">
<div class="column" style="font-size: 75%;">
<ul>
<li><p>Dia 1 - Manhã:</p>
<ul>
<li><p>Apresentação;</p></li>
<li><p>Sequenciamento nanopore;</p></li>
<li><p>Instalação dos softwares;</p></li>
</ul></li>
</ul>
<p><br></p>
<ul>
<li><p>Dia 1 - Tarde:</p>
<ul>
<li><p>Chamada de base;</p></li>
<li><p>Prática;</p></li>
</ul></li>
</ul>
<p><br></p>
</div><div class="column" style="font-size: 75%;">
<ul>
<li><p>Dia 2 - Manhã:</p>
<ul>
<li><p>Mapeamento de sequências;</p></li>
<li><p>Prática;</p></li>
</ul></li>
</ul>
<p><br></p>
<ul>
<li><p>Dia 2 - Tarde:</p>
<ul>
<li><p>Chamada de variantes;</p></li>
<li><p>Prática;</p></li>
</ul></li>
</ul>
</div></div>
</section>
<section id="cronograma-1" class="slide level2">
<h2>Cronograma</h2>
<div class="columns">
<div class="column" style="font-size: 75%;">
<ul>
<li><p>Dia 3 - Manhã:</p>
<ul>
<li><p>Variantes estruturais;</p></li>
<li><p>Prática;</p></li>
</ul></li>
</ul>
<p><br></p>
<ul>
<li><p>Dia 3 - Tarde:</p>
<ul>
<li><p>Determinação de haplótipos;</p></li>
<li><p>Prática;</p></li>
</ul></li>
</ul>
<p><br></p>
</div><div class="column" style="font-size: 75%;">
<ul>
<li><p>Dia 4 - Manhã:</p>
<ul>
<li><p>Montagem de genomas;</p></li>
<li><p>Prática;</p></li>
</ul></li>
</ul>
<p><br></p>
<ul>
<li><p>Dia 4 - Tarde:</p>
<ul>
<li><p>Modificação de bases;</p></li>
<li><p>Prática;</p>
<p><br></p></li>
<li><p>Adeus 😥.</p></li>
</ul></li>
</ul>
</div></div>
</section>
<section id="avaliação" class="slide level2" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Avaliação</h2>
<ul>
<li>3 provas;</li>
<li>2 seminários;</li>
<li>Estudo dirigido;</li>
</ul>
</section>
<section id="avaliação-1" class="slide level2" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Avaliação</h2>
<ul>
<li><del>3 provas;</del></li>
<li><del>2 seminários;</del></li>
<li><del>Estudo dirigido;</del></li>
</ul>
<p><br></p>
<ul>
<li>Participação na disciplina e fazer as atividades.</li>
</ul>
</section>
<section id="programas-e-dados" class="slide level2">
<h2>Programas e dados</h2>
<ul>
<li>Baixar arquivos do Drive [<a href="https://drive.google.com/drive/folders/1mkG-atQJBop-nr5qbk25UpRNxi8YzC6c?usp=drive_link">link</a>];</li>
</ul>
<p><br></p>
<ul>
<li>Instalar o IGV <span class="citation" data-cites="robinson2011">(<a href="#/referências" role="doc-biblioref" onclick="">Robinson et al. 2011</a>)</span> [<a href="https://igv.org/doc/desktop/">link</a>];</li>
</ul>
<p><br></p>
<ul>
<li>Usar o script de instalação dos programas no Linux;</li>
</ul>
</section></section>
<section>
<section id="sequenciamento-via-ont" class="title-slide slide level1 center">
<h1>Sequenciamento via ONT</h1>

</section>
<section id="ordens-de-grandeza-do-genoma-humano" class="slide level2">
<h2>Ordens de Grandeza do Genoma Humano</h2>
<ul>
<li><p>Número de cromossomos: 46 (23 pares);</p></li>
<li><p>Tamanho do genoma haploide: ~3,2 bilhões de pares de bases (3,2 Gb);</p></li>
<li><p>Número estimado de genes codificadores de proteínas: ~20.000;</p></li>
</ul>
</section>
<section id="section" class="slide level2">
<h2></h2>

<img data-src="humanchr.png" class="quarto-figure quarto-figure-center r-stretch"></section>
<section id="por-que-sequenciar" class="slide level2">
<h2>Por que sequenciar?</h2>
<div class="fragment">
<ul>
<li><p>A sequência é a “receita” da vida</p>
<ul>
<li>Ela determina a estrutura e função das moléculas biológicas.</li>
</ul></li>
<li><p><strong>DNA</strong> → <strong>RNA</strong> → <strong>Proteína</strong></p>
<ul>
<li>Alterações na sequência podem afetar a função celular e causar doenças.</li>
</ul></li>
<li><p>Entender a sequência = entender o funcionamento dos organismos</p>
<ul>
<li>Do gene à característica observável (fenótipo).</li>
</ul></li>
<li><p>Permite identificar diferenças genéticas</p>
<ul>
<li>Entre indivíduos, espécies, populações ou células (ex: câncer).</li>
</ul></li>
</ul>
</div>
</section>
<section id="geração-de-sequenciamento" class="slide level2">
<h2>Geração de sequenciamento</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://pit.edu.vn/en/post/generations-of-gene-sequencing-technology-139.htm"><img data-src="seqgens.png" alt="https://pit.edu.vn/en/post/generations-of-gene-sequencing-technology-139.htm"></a></p>
<figcaption>https://pit.edu.vn/en/post/generations-of-gene-sequencing-technology-139.htm</figcaption>
</figure>
</div>
</section>
<section id="sequenciamento-comparativo-entre-plataformas" class="slide level2">
<h2>Sequenciamento: Comparativo entre Plataformas</h2>
<p><br></p>
<div style="font-size:80%;">
<table class="caption-top" style="width:100%;">
<colgroup>
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
</colgroup>
<tbody>
<tr class="odd">
<td><strong>Plataforma</strong></td>
<td><strong>Tipo de Leitura</strong></td>
<td><strong>Tamanho de Leitura</strong></td>
<td><strong>Taxa de Erro</strong></td>
<td><strong>Tempo de Execução</strong></td>
<td><strong>Aplicações Comuns</strong></td>
</tr>
<tr class="even">
<td>Illumina</td>
<td>Short reads</td>
<td>150–300 pares de bases</td>
<td>&lt;1%</td>
<td>1–2 dias</td>
<td>RNA-seq, exoma, WGS, genotipagem</td>
</tr>
<tr class="odd">
<td>PacBio Hifi</td>
<td>Long reads (alta fidelidade)</td>
<td>10–25 kb (HiFi)</td>
<td>~1%</td>
<td>1–2 dias</td>
<td>Montagem genômica, haplótipos, variantes estruturais</td>
</tr>
<tr class="even">
<td>Oxford Nanopre</td>
<td>Long/ultralong reads</td>
<td>10 kb – &gt;1 Mb</td>
<td>5–10% (melhorando)</td>
<td>Horas a 2 dias</td>
<td>Metagenômica, epigenética, transcriptômica, forense</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="introdução-a-tecnologia-de-sequenciamento" class="slide level2">
<h2>Introdução a tecnologia de sequenciamento</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.genome.gov/geneticsglossary/DNASequencing#:~:text=DNA%20sequencing%20refers%20to%20the,use%20to%20develop%20and%20operate."><img data-src="seqmethods.png" class="quarto-figure quarto-figure-center" alt="https://www.genome.gov/geneticsglossary/DNASequencing#:~:text=DNA%20sequencing%20refers%20to%20the,use%20to%20develop%20and%20operate."></a></p>
</figure>
</div>
<figcaption>https://www.genome.gov/geneticsglossary/DNASequencing#:~:text=DNA%20sequencing%20refers%20to%20the,use%20to%20develop%20and%20operate.</figcaption>
</figure>
</div>
</section>
<section id="oxford-nanopore-technologies-ont" class="slide level2">
<h2>Oxford nanopore technologies (ONT)</h2>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-left">
<figure>
<p><img data-src="equipaments.png" class="quarto-figure quarto-figure-left"></p>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%;">
<ul>
<li><p>Portátil e escalável;</p></li>
<li><p>“Barato”;</p></li>
<li><p>Aquisição de dados em tempo real;</p></li>
<li><p>Altos volumes de dados (fastq &gt; 50 Gb);</p></li>
<li><p>DNA* e RNA*;</p></li>
<li><p>Long reads (10 kb – 100 Kb);</p></li>
<li><p>Ultra (100 Kb – 300 Kb);</p></li>
<li><p>Recorde 4 Mb!!!!!!;</p></li>
<li><p>Acurácia atual de &gt;99%;</p></li>
</ul>
</div>
</div></div>
</div>
</section>
<section id="ont---flowcells" class="slide level2">
<h2>ONT - Flowcells</h2>

<img data-src="flowcells.png" class="r-stretch quarto-figure-center"><p class="caption">Valores teórico de capacidade de sequenciamento.</p></section>
<section id="ont---princípio" class="slide level2">
<h2>ONT - Princípio</h2>

<img data-src="ont_principle.png" class="r-stretch quarto-figure-center"><p class="caption"><span class="citation" data-cites="wang2021">(<a href="#/referências" role="doc-biblioref" onclick="">Wang et al. 2021</a>)</span></p></section>
<section id="ont---princípio-1" class="slide level2">
<h2>ONT - Princípio</h2>
<iframe data-external="1" src="https://www.youtube.com/embed/RcP85JHLmnI?si=tmBwQYEmgRZa34zA" width="90%" height="80%" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
</section>
<section id="ont---nanoporos" class="slide level2">
<h2>ONT - Nanoporos</h2>

<img data-src="napores.png" class="r-stretch quarto-figure-center"><p class="caption">Lu, C., Bonini, A., Viel, J.H.&nbsp;et al.&nbsp;Toward single-molecule protein sequencing using nanopores.&nbsp;Nat Biotechnol&nbsp;43, 312–322 (2025). <span class="citation" data-cites="Lu2025">(<a href="#/referências" role="doc-biblioref" onclick="">Lu et al. 2025</a>)</span></p></section>
<section id="ont---tipos-de-leitura" class="slide level2">
<h2>ONT - Tipos de leitura</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/document/kit-14-device-and-informatics"><img data-src="duplex.webp" class="quarto-figure quarto-figure-center" width="900" alt="Duplex."></a></p>
</figure>
</div>
<figcaption>Duplex.</figcaption>
</figure>
</div>
</section>
<section id="ont---tipos-de-leitura-1" class="slide level2">
<h2>ONT - Tipos de leitura</h2>
<ul>
<li><p><code>Simplex</code>: O sequenciamento de uma única fita. A fita de DNA modelo passa pelo nanoporo e é submetida ao basecalling. Isso é realizado no MinKNOW.</p></li>
<li><p><code>Duplex</code>: O sequenciamento de ambas as fitas. A fita complementar é lida imediatamente após a fita modelo e o basecalling consensual para ambas as fitas leva a um aumento ainda maior na precisão. Isso é realizado no Dorado.</p></li>
</ul>

<aside><div>
<p>MinKNOW é o software que controla o sequenciamento.</p>
</div></aside></section>
<section id="ont---leitura-da-fita" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="leituradafita.png"></p>
<figcaption>Sutton, M.A., Burton, A.S., Zaikova, E. et al.&nbsp;Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%">

</div>
</div></div>
</section>
<section id="ont---leitura-da-fita-1" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="leituradafita.png"></p>
<figcaption>Sutton, M.A., Burton, A.S., Zaikova, E. et al.&nbsp;Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%">
<ul>
<li><p>Leitura DNA = 400 bp por segundo;</p></li>
<li><p>RNA = 70 bp por segundo;</p></li>
<li><p>6 bases simultâneas por poro;</p></li>
<li><p>{A,T,C,G};</p></li>
</ul>
</div>
</div></div>
</section>
<section id="ont---leitura-da-fita-2" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="leituradafita.png"></p>
<figcaption>Sutton, M.A., Burton, A.S., Zaikova, E. et al.&nbsp;Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%">
<ul>
<li><p>Leitura DNA = 400 bp por segundo;</p></li>
<li><p>RNA = 70 bp por segundo;</p></li>
<li><p>6 bases simultâneas por poro;</p></li>
<li><p>{A,T,C,G};</p></li>
</ul>
<p><br></p>
<ul>
<li>4x4x4x4x4x4 =?</li>
</ul>
</div>
</div></div>
</section>
<section id="ont---leitura-da-fita-3" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="leituradafita.png"></p>
<figcaption>Sutton, M.A., Burton, A.S., Zaikova, E. et al.&nbsp;Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%">
<ul>
<li><p>Leitura DNA = 400 bp por segundo;</p></li>
<li><p>RNA = 70 bp por segundo;</p></li>
<li><p>6 bases simultâneas por poro;</p></li>
<li><p>{A,T,C,G};</p></li>
</ul>
<p><br></p>
<ul>
<li>4x4x4x4x4x4 =?</li>
</ul>
<p><br></p>
<ul>
<li>4096 combinações!</li>
</ul>
</div>
</div></div>
</section>
<section id="ont---leitura-da-fita-4" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="center">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/technology/basecalling"><img data-src="https://a.storyblok.com/f/196663/5384x5511/5d1ab2a959/different-types-of-nanopore.png" width="488" alt="Modelos de poros tem dois centros de leitura."></a></p>
<figcaption>Modelos de poros tem dois centros de leitura.</figcaption>
</figure>
</div>
</div>
</section>
<section id="ont---rna" class="slide level2">
<h2>ONT - RNA</h2>

<img data-src="rna.png" class="quarto-figure quarto-figure-center r-stretch" width="1300"></section>
<section id="modificações-no-dna" class="slide level2">
<h2>Modificações no DNA</h2>
<div class="fragment">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="ont_prinmods.png" width="806"></p>
<figcaption>https://www.nature.com/articles/s41587-021-01108-x</figcaption>
</figure>
</div>
</div>
</section>
<section id="modificações-no-dna-1" class="slide level2">
<h2>Modificações no DNA</h2>
<ul>
<li>Existem cerca de 30 modificações de base descrita!</li>
</ul>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<p>{A, T, C, G, 5mC}</p>
</div><div class="column" style="width:50%;">
<p>{A, T, C, G, 5mC, 5hmC}</p>
</div></div>
</div>
</section>
<section id="modificações-no-dna-2" class="slide level2">
<h2>Modificações no DNA</h2>
<ul>
<li>Existem cerca de 30 modificações de base descrita!</li>
</ul>
<div class="columns">
<div class="column" style="width:50%;">
<p>{A, T, C, G, 5mC}</p>
<p><br></p>
<p>5x5x5x5x5x5 = 15625 combinações!</p>
</div><div class="column" style="width:50%;">
<p>{A, T, C, G, 5mC, 5hmC}</p>
<p><br></p>
<p>6x6x6x6x6x6 = 46656 combinações!</p>
</div></div>
</section>
<section id="basecallers" class="slide level2">
<h2>Basecallers</h2>

<img data-src="images/ont_geral.png" width="1200" class="r-stretch quarto-figure-center"><p class="caption">Basecalling.</p></section>
<section id="basecallers-1" class="slide level2">
<h2>Basecallers</h2>
<ul>
<li><p>Nanoporos diferentes contêm “leitores” diferentes;</p>
<ul>
<li>R10.4.1 é o modelo atual;</li>
</ul></li>
<li><p>Capturam o sinal elétrico (POD5);</p></li>
<li><p>Basecallers transformam POD5 em FASTQ;</p></li>
<li><p>Utilizam Machine Learning (RNN);</p></li>
<li><p>Este processo pode ser feito em tempo real;</p></li>
</ul>

<aside><div>
<p>POD5 é o formato atual, mas pode-se encontrar FAST5;</p>
</div></aside></section>
<section id="basecallers-2" class="slide level2">
<h2>Basecallers</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/technology/basecalling"><img data-src="https://a.storyblok.com/f/196663/809x716/e4743867e2/neural-networks.png" class="quarto-figure quarto-figure-center" width="565" alt="Como funcionam"></a></p>
</figure>
</div>
<figcaption>Como funcionam</figcaption>
</figure>
</div>
</section>
<section id="basecallers-3" class="slide level2">
<h2>Basecallers</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/technology/basecalling"><img data-src="https://a.storyblok.com/f/196663/2326x960/ae453e7f75/basecallers-timeline.png" class="quarto-figure quarto-figure-center" width="1200" alt="Evolução dos modelos."></a></p>
</figure>
</div>
<figcaption>Evolução dos modelos.</figcaption>
</figure>
</div>
</section>
<section id="basecallers-4" class="slide level2">
<h2>Basecallers</h2>
<p>Sugestão de leitura:</p>
<ul>
<li><em>“<strong>From squiggle to basepair: computational approaches for improving nanopore sequencing read accuracy</strong>”</em> <span class="citation" data-cites="rang2018">(<a href="#/referências" role="doc-biblioref" onclick="">Rang, Kloosterman, e Ridder 2018</a>)</span>[<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1462-9">link</a>]</li>
</ul>
</section>
<section id="bancada" class="slide level2">
<h2>Bancada</h2>
<div class="fragment">
<ul>
<li>hmwDNA;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>Alta pureza;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>Não há necessidade de PCR;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>Não há necessidade conversões por bissulfito de sódio (e afins);</li>
</ul>
</div>
</section>
<section id="enriquecimento-do-sequenciamento" class="slide level2">
<h2>Enriquecimento do sequenciamento</h2>
<div class="fragment">
<ul>
<li><strong><em>Target</em></strong>;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li><strong><em>CRISPR</em></strong>;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li><strong><em>Adaptive sampling</em></strong>;</li>
</ul>
</div>
</section>
<section id="enriquecimento-do-sequenciamento-1" class="slide level2">
<h2>Enriquecimento do sequenciamento</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1872497324001522"><img data-src="images/enrichment3.png" class="quarto-figure quarto-figure-center" width="1200" alt="Estratégias de enriquecimento de ONT"></a></p>
</figure>
</div>
<figcaption>Estratégias de enriquecimento de ONT</figcaption>
</figure>
</div>
</section>
<section id="potencial" class="slide level2">
<h2>Potencial</h2>
</section>
<section id="potencial---montagem-de-genomas" class="slide level2">
<h2>Potencial - montagem de genomas</h2>

<img data-src="montagemdegenomas.png" width="1000" class="r-stretch quarto-figure-center"><p class="caption">https://www.phgfoundation.org/briefing/clinical-long-read-sequencing</p></section>
<section id="potencial---montagem-de-genomas-1" class="slide level2">
<h2>Potencial - montagem de genomas</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.science.org/doi/10.1126/science.abj6987"><img data-src="completehumangenome.png" class="quarto-figure quarto-figure-center" width="596" alt="Montagem completa do genoma humano."></a></p>
</figure>
</div>
<figcaption>Montagem completa do genoma humano.</figcaption>
</figure>
</div>

<aside><div>
<p><span class="citation" data-cites="nurk2022">(<a href="#/referências" role="doc-biblioref" onclick="">Nurk et al. 2022</a>)</span></p>
</div></aside></section>
<section id="potencial---montagem-de-genomas-2" class="slide level2">
<h2>Potencial - montagem de genomas</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.science.org/cms/10.1126/science.abj6987/asset/ec796ff1-289e-43cf-a065-ec6573199a8d/assets/images/large/science.abj6987-f1.jpg"><img data-src="completehumangenome_2.png" class="quarto-figure quarto-figure-center" width="393" alt="Aumento de bases no genoma."></a></p>
</figure>
</div>
<figcaption>Aumento de bases no genoma.</figcaption>
</figure>
</div>
</section>
<section id="potencial---haplótipos-longos" class="slide level2">
<h2>Potencial - haplótipos longos</h2>

<img data-src="longhap.png" width="1200" class="r-stretch quarto-figure-center"><p class="caption">Determinar haplótipos longos</p></section>
<section id="potencial---haplótipos-longos-1" class="slide level2">
<h2>Potencial - haplótipos longos</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.nature.com/articles/s41467-024-47349-7/figures/4"><img data-src="pecat.webp" class="quarto-figure quarto-figure-center" width="797" alt="Haplótipos do pai e da mãe."></a></p>
</figure>
</div>
<figcaption>Haplótipos do pai e da mãe.</figcaption>
</figure>
</div>
</section>
<section id="potencial---metilação" class="slide level2">
<h2>Potencial - metilação</h2>

<img data-src="hapmet.png" width="500" class="r-stretch quarto-figure-center"><p class="caption">Metilação ao longo do gene MYEF2.</p></section>
<section id="potencial---outros" class="slide level2">
<h2>Potencial - outros</h2>
<ul>
<li>STR <span class="citation" data-cites="luo2024">(<a href="#/referências" role="doc-biblioref" onclick="">Luo et al. 2024</a>)</span>;</li>
</ul>
<p><br></p>
<ul>
<li>Elementos transponíveis <span class="citation" data-cites="smits2022">(<a href="#/referências" role="doc-biblioref" onclick="">Smits e Faulkner 2022</a>)</span>;</li>
</ul>
<p><br></p>
<ul>
<li>Epitranscriptômica <span class="citation" data-cites="Workman2019 Begik2022 Jain2022">(<a href="#/referências" role="doc-biblioref" onclick="">Workman et al. 2019</a>; <a href="#/referências" role="doc-biblioref" onclick="">Begik, Mattick, e Novoa 2022</a>; <a href="#/referências" role="doc-biblioref" onclick="">Jain et al. 2022</a>)</span>;</li>
</ul>
</section>
<section id="estado-atual" class="slide level2">
<h2>Estado atual</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/accuracy"><img data-src="https://a.storyblok.com/f/196663/10800x3600/86aa012b7e/20250613_dna_q-1.png" class="quarto-figure quarto-figure-center" width="1200" alt="Sequenciamento de DNA"></a></p>
</figure>
</div>
<figcaption>Sequenciamento de DNA</figcaption>
</figure>
</div>
<p><span class="math display">\[
P = 10^{\frac{-Q}{10}}
\]</span></p>
</section>
<section id="estado-atual-1" class="slide level2">
<h2>Estado atual</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/accuracy"><img data-src="https://a.storyblok.com/f/196663/1158x952/4cc71d5637/base-modification-section-graph_mod.jpg" class="quarto-figure quarto-figure-center" width="608" alt="Metilação."></a></p>
</figure>
</div>
<figcaption>Metilação.</figcaption>
</figure>
</div>
</section>
<section id="epi2me" class="slide level2">
<h2>EPI2ME</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li><p><a href="https://epi2me.nanoporetech.com/wfindex/">EPI2ME</a> é uma coleção de pipelines fornecidos pela própria ONT;</p></li>
<li><p>Nextflow;</p></li>
<li><p>Útil para quem quer ou não programar;</p></li>
</ul>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://epi2me.nanoporetech.com/wfindex/"><img data-src="https://epi2me.nanoporetech.com/static/dbb47454a74ffdac6ab079de768cc8af/3421a/logo-dark.webp" class="quarto-figure quarto-figure-center" width="450"></a></p>
</figure>
</div>
</div></div>
</section>
<section id="futuro-da-tecnologia" class="slide level2">
<h2>Futuro da tecnologia</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.nature.com/articles/s41587-025-02587-y#citeas"><img data-src="protein.png" class="quarto-figure quarto-figure-center" alt="Lu, C., Bonini, A., Viel, J.H. et al.&nbsp;Toward single-molecule protein sequencing using nanopores. Nat Biotechnol 43, 312–322 (2025). https://doi.org/10.1038/s41587-025-02587-y"></a></p>
</figure>
</div>
<figcaption>Lu, C., Bonini, A., Viel, J.H. et al.&nbsp;Toward single-molecule protein sequencing using nanopores. Nat Biotechnol 43, 312–322 (2025). https://doi.org/10.1038/s41587-025-02587-y</figcaption>
</figure>
</div>
</section>
<section id="futuro-da-tecnologia-1" class="slide level2">
<h2>Futuro da tecnologia</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://comics.dcv.fct.unl.pt/resources/"><img data-src="multiomics.png" class="quarto-figure quarto-figure-center" alt="https://comics.dcv.fct.unl.pt/resources/"></a></p>
</figure>
</div>
<figcaption>https://comics.dcv.fct.unl.pt/resources/</figcaption>
</figure>
</div>
</section>
<section id="desafios-e-frustrações.." class="slide level2">
<h2>Desafios (e frustrações)..</h2>
<div style="text-align: center;">
<iframe src="https://giphy.com/embed/2zdVnqmhSfAvyQhzHj" width="480" height="480" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/showtime-season-4-episode-2zdVnqmhSfAvyQhzHj">via GIPHY</a>
</p>
</div>
</section>
<section id="desafios-e-frustrações..-1" class="slide level2">
<h2>Desafios (e frustrações)..</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://github.com/rrwick"><img data-src="https://avatars.githubusercontent.com/u/7053555?v=4.png" width="418" alt="Ryan Wick"></a></p>
<figcaption>Ryan Wick</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 90%">
<p><strong><em>“May 2025 update</em></strong></p>
<p><em>I’ve recently heard that ONT is deprecating duplex basecalling – not surprising given their recent silence on the topic. This is now the third time (after 2D and 1D<sup>2</sup>) that ONT has tried and dropped both-strand basecalling! So it seems that mixed simplex-duplex read sets like the ones in this post will end up a historical curiosity rather than a standard part of ONT sequencing.”</em> [<a href="https://rrwick.github.io/2024/05/08/duplex_assemblies.html">link</a>]</p>
</div>
</div></div>
</section>
<section id="visão-sobre-a-tecnologia" class="slide level2">
<h2>Visão sobre a tecnologia</h2>
</section>
<section id="visão-sobre-a-tecnologia-1" class="slide level2" data-background-image="genildo.png" data-background-opacity=".25">
<h2>Visão sobre a tecnologia</h2>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<div style="color: black; font-size: 140%; font-weight: bold; text-shadow: 2px 2px 4px black;">
<p><br></p>
<ul>
<li><p>Alterações constantes;</p></li>
<li><p>Precisa sempre adaptar;</p></li>
<li><p>Muitas opções de pipeline;</p></li>
<li><p>Diagnóstico 👎;</p></li>
</ul>
</div>
</div><div class="column" style="width:50%;">
<div style="color: black; font-size: 140%; font-weight: bold; text-shadow: 2px 2px 4px black;">
<p><br></p>
<ul>
<li><p>Melhorias constantes;</p></li>
<li><p>Somos pesquisadores;</p></li>
<li><p>Comunidade e suporte;</p></li>
<li><p>Pesquisa 👍;</p></li>
</ul>
</div>
</div></div>
</div>
</section></section>
<section>
<section id="dados-para-essa-disciplina" class="title-slide slide level1 center">
<h1>Dados para essa disciplina</h1>

</section>
<section id="amostras" class="slide level2">
<h2>Amostras</h2>
<ul>
<li><p>As amostras deste disciplinas são referentes ao projeto 1000 genomas <span class="citation" data-cites="gustafson2024">(<a href="#/referências" role="doc-biblioref" onclick="">Gustafson et al. 2024</a>)</span>. [<a href="https://s3.amazonaws.com/1000g-ont/index.html?prefix=ALIGNMENT_AND_ASSEMBLY_DATA/100_PLUS/IN-HOUSE_MINIMAP2/">link</a>]</p></li>
<li><p>Os códigos das amostras são:</p>
<ul>
<li><p><a href="https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG00109">HG00109</a>;</p></li>
<li><p><a href="https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG00463&amp;Product=DNA">HG00463</a>;</p></li>
<li><p><a href="https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG00663&amp;Product=CC">HG00663</a>;</p></li>
<li><p><a href="https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG01063&amp;PgId=166">HG01063</a>;</p></li>
<li><p><a href="https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG01303&amp;Product=CC">HG01303</a>;</p></li>
<li><p><a href="https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG01524">HG01524</a>;</p></li>
</ul></li>
</ul>
</section>
<section id="genes" class="slide level2">
<h2>Genes</h2>
<div class="cell">
<div class="cell-output-display">
<div id="hemqmqsdeh" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#hemqmqsdeh table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#hemqmqsdeh thead, #hemqmqsdeh tbody, #hemqmqsdeh tfoot, #hemqmqsdeh tr, #hemqmqsdeh td, #hemqmqsdeh th {
  border-style: none;
}

#hemqmqsdeh p {
  margin: 0;
  padding: 0;
}

#hemqmqsdeh .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 30px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 80%;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#hemqmqsdeh .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#hemqmqsdeh .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#hemqmqsdeh .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#hemqmqsdeh .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#hemqmqsdeh .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#hemqmqsdeh .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#hemqmqsdeh .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#hemqmqsdeh .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#hemqmqsdeh .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#hemqmqsdeh .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#hemqmqsdeh .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#hemqmqsdeh .gt_spanner_row {
  border-bottom-style: hidden;
}

#hemqmqsdeh .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#hemqmqsdeh .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#hemqmqsdeh .gt_from_md > :first-child {
  margin-top: 0;
}

#hemqmqsdeh .gt_from_md > :last-child {
  margin-bottom: 0;
}

#hemqmqsdeh .gt_row {
  padding-top: 12px;
  padding-bottom: 12px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#hemqmqsdeh .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#hemqmqsdeh .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#hemqmqsdeh .gt_row_group_first td {
  border-top-width: 2px;
}

#hemqmqsdeh .gt_row_group_first th {
  border-top-width: 2px;
}

#hemqmqsdeh .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#hemqmqsdeh .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#hemqmqsdeh .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#hemqmqsdeh .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#hemqmqsdeh .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#hemqmqsdeh .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#hemqmqsdeh .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#hemqmqsdeh .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#hemqmqsdeh .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#hemqmqsdeh .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#hemqmqsdeh .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#hemqmqsdeh .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#hemqmqsdeh .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#hemqmqsdeh .gt_left {
  text-align: left;
}

#hemqmqsdeh .gt_center {
  text-align: center;
}

#hemqmqsdeh .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#hemqmqsdeh .gt_font_normal {
  font-weight: normal;
}

#hemqmqsdeh .gt_font_bold {
  font-weight: bold;
}

#hemqmqsdeh .gt_font_italic {
  font-style: italic;
}

#hemqmqsdeh .gt_super {
  font-size: 65%;
}

#hemqmqsdeh .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#hemqmqsdeh .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#hemqmqsdeh .gt_indent_1 {
  text-indent: 5px;
}

#hemqmqsdeh .gt_indent_2 {
  text-indent: 10px;
}

#hemqmqsdeh .gt_indent_3 {
  text-indent: 15px;
}

#hemqmqsdeh .gt_indent_4 {
  text-indent: 20px;
}

#hemqmqsdeh .gt_indent_5 {
  text-indent: 25px;
}
</style>

<table class="gt_table caption-top" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_col_headings header">
<th id="chr" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">chr</th>
<th id="start" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">start</th>
<th id="end" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">end</th>
<th id="gene" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">gene</th>
<th id="strand" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">strand</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_center" headers="chr">chr1</td>
<td class="gt_row gt_center" headers="start">1434861</td>
<td class="gt_row gt_center" headers="end">1442882</td>
<td class="gt_row gt_center" headers="gene">VWA1</td>
<td class="gt_row gt_center" headers="strand">+</td>
</tr>
<tr class="even">
<td class="gt_row gt_center" headers="chr">chr15</td>
<td class="gt_row gt_center" headers="start">27754875</td>
<td class="gt_row gt_center" headers="end">28099315</td>
<td class="gt_row gt_center" headers="gene">OCA2</td>
<td class="gt_row gt_center" headers="strand">-</td>
</tr>
<tr class="odd">
<td class="gt_row gt_center" headers="chr">chr15</td>
<td class="gt_row gt_center" headers="start">98648539</td>
<td class="gt_row gt_center" headers="end">98964530</td>
<td class="gt_row gt_center" headers="gene">IGF1R</td>
<td class="gt_row gt_center" headers="strand">+</td>
</tr>
<tr class="even">
<td class="gt_row gt_center" headers="chr">chr16</td>
<td class="gt_row gt_center" headers="start">68737292</td>
<td class="gt_row gt_center" headers="end">68835537</td>
<td class="gt_row gt_center" headers="gene">CDH1</td>
<td class="gt_row gt_center" headers="strand">+</td>
</tr>
<tr class="odd">
<td class="gt_row gt_center" headers="chr">chr17</td>
<td class="gt_row gt_center" headers="start">7661779</td>
<td class="gt_row gt_center" headers="end">7687546</td>
<td class="gt_row gt_center" headers="gene">TP53</td>
<td class="gt_row gt_center" headers="strand">-</td>
</tr>
<tr class="even">
<td class="gt_row gt_center" headers="chr">chr22</td>
<td class="gt_row gt_center" headers="start">45920366</td>
<td class="gt_row gt_center" headers="end">45977162</td>
<td class="gt_row gt_center" headers="gene">WNT7B</td>
<td class="gt_row gt_center" headers="strand">-</td>
</tr>
<tr class="odd">
<td class="gt_row gt_center" headers="chr">chr6</td>
<td class="gt_row gt_center" headers="start">45328157</td>
<td class="gt_row gt_center" headers="end">45664349</td>
<td class="gt_row gt_center" headers="gene">RUNX2</td>
<td class="gt_row gt_center" headers="strand">+</td>
</tr>
<tr class="even">
<td class="gt_row gt_center" headers="chr">chr7</td>
<td class="gt_row gt_center" headers="start">27153716</td>
<td class="gt_row gt_center" headers="end">27157936</td>
<td class="gt_row gt_center" headers="gene">HOXA7</td>
<td class="gt_row gt_center" headers="strand">-</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
</section>
<section id="programas" class="slide level2">
<h2>Programas</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li>Linux;</li>
</ul>
<p><br></p>
<ul>
<li>Usuários windows podem usar WSL;</li>
</ul>
<p><br></p>
<ul>
<li>Script para instalação;</li>
</ul>
</div><div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://anaconda.org/anaconda/conda"><img data-src="https://raw.githubusercontent.com/devicons/devicon/54cfe13ac10eaa1ef817a343ab0a9437eb3c2e08/icons/anaconda/anaconda-original-wordmark.svg" class="quarto-figure quarto-figure-center" width="350" alt="Conda"></a></p>
</figure>
</div>
<figcaption>Conda</figcaption>
</figure>
</div>
</div></div>
</section>
<section id="o-que-é-conda" class="slide level2">
<h2>O que é Conda?</h2>
<div style="text-align: center;">
<iframe src="https://giphy.com/embed/WdC26ECYnvTX4Mp2fI" width="480" height="480" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/primevideo-lularich-WdC26ECYnvTX4Mp2fI">via GIPHY</a>
</p>
</div>
</section>
<section id="conda" class="slide level2">
<h2>Conda</h2>
<ul>
<li>Gerenciador de pacotes/programas;</li>
</ul>
<p><br></p>
<ul>
<li>Cria ambiente reprodutíveis;</li>
</ul>
</section></section>
<section>
<section id="chamada-de-base-basecalling" class="title-slide slide level1 center">
<h1>Chamada de base (basecalling)</h1>

</section>
<section id="chamada-de-base" class="slide level2">
<h2>Chamada de base</h2>
<ul>
<li><p>Procedimento no qual o sinal elétrico (<code>.pod5</code>) é convertido em bases (<code>.fastq</code> | <code>.bam</code>);</p></li>
<li><p>Momento crucial onde se define o perfil da análise que será realizada;</p></li>
<li><p>Erros aqui prejudicam todas as demais análises;</p></li>
<li><p><code>Dorado</code>;</p></li>
</ul>
</section>
<section id="qual-modelo-usar" class="slide level2">
<h2>Qual modelo usar?</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb1-1"><a href=""></a><span class="ex">{analyte}_{pore}_{chemistry}_{speed}@version</span></span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li><p><code>analyte</code>: DNA ou RNA?</p></li>
<li><p><code>pore</code>: Qual técnologia de poro? (<code>R10.4.1</code>/<code>R9.4.1</code>)</p></li>
<li><p><code>chemistry</code>: Qual kit foi utilizado foi utilizado?</p></li>
<li><p><code>speed</code>: Velocidade de leitura durante o sequenciamento (DNA 400bps)</p></li>
<li><p><code>version</code>: Versão do modelo</p></li>
</ul>
</section>
<section id="comando-dorado" class="slide level2">
<h2>Comando <code>dorado</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb2-1"><a href=""></a><span class="op">&gt;</span> dorado <span class="ex">--help</span></span>
<span id="cb2-2"><a href=""></a><span class="ex">Usage:</span> dorado <span class="pp">[</span><span class="ss">options</span><span class="pp">]</span> subcommand</span>
<span id="cb2-3"><a href=""></a></span>
<span id="cb2-4"><a href=""></a><span class="ex">Positional</span> arguments:</span>
<span id="cb2-5"><a href=""></a><span class="ex">aligner</span></span>
<span id="cb2-6"><a href=""></a><span class="ex">basecaller</span></span>
<span id="cb2-7"><a href=""></a><span class="ex">correct</span></span>
<span id="cb2-8"><a href=""></a><span class="ex">demux</span></span>
<span id="cb2-9"><a href=""></a><span class="ex">download</span></span>
<span id="cb2-10"><a href=""></a><span class="ex">duplex</span></span>
<span id="cb2-11"><a href=""></a><span class="ex">summary</span></span>
<span id="cb2-12"><a href=""></a><span class="ex">trim</span></span>
<span id="cb2-13"><a href=""></a></span>
<span id="cb2-14"><a href=""></a><span class="ex">Optional</span> arguments:</span>
<span id="cb2-15"><a href=""></a><span class="ex">-h</span> <span class="at">--help</span>               shows help message and exits</span>
<span id="cb2-16"><a href=""></a><span class="ex">-v</span> <span class="at">--version</span>            prints version information and exits</span>
<span id="cb2-17"><a href=""></a><span class="ex">-vv</span>                     prints verbose version information and exits</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="modelos-dorado" class="slide level2">
<h2>Modelos <code>dorado</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb3-1"><a href=""></a><span class="co"># Ver os modelos disponíveis</span></span>
<span id="cb3-2"><a href=""></a><span class="ex">dorado</span> download <span class="at">--list</span></span>
<span id="cb3-3"><a href=""></a></span>
<span id="cb3-4"><a href=""></a><span class="co"># Baixar o desejado na pasta desejada</span></span>
<span id="cb3-5"><a href=""></a><span class="ex">dorado</span> download <span class="at">--model</span> {modelo} <span class="at">--models-directory</span> {PATH}</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="modelos-dorado-1" class="slide level2">
<h2>Modelos <code>dorado</code></h2>
<ul>
<li><p><code>FAST</code>: Mais rápido e com menor acurácia;</p></li>
<li><p><code>HAC</code>: Alta acurácia;</p></li>
<li><p><code>SUP</code>: Super acurácia (maior requisito computacional);</p></li>
</ul>
<div class="callout callout-tip callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Para escolher seu modelo.</strong></p>
</div>
<div class="callout-content">
<p>Modelo <code>SUP</code> é uma vantagem em situações de construção de genomas de referência;</p>
<p>Para demais situações o <code>HAC</code> já é suficiente;</p>
</div>
</div>
</div>
</section>
<section id="dorado-basecaller" class="slide level2">
<h2><code>dorado</code> <code>basecaller</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb4-1"><a href=""></a><span class="co"># chamada HAC sem modificações</span></span>
<span id="cb4-2"><a href=""></a><span class="ex">dorado</span> basecaller hac {/path/to/reads.pod5} <span class="op">&gt;</span> calls.bam</span>
<span id="cb4-3"><a href=""></a></span>
<span id="cb4-4"><a href=""></a><span class="co">#chamada HAC com modificações 6mA e 5mC e 5hmC em CpG</span></span>
<span id="cb4-5"><a href=""></a><span class="ex">dorado</span> basecaller hac,6mA,5mCG_5hmCG {/path/to/reads.pod5} <span class="op">&gt;</span> calls.bam</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="dorado-basecaller-1" class="slide level2">
<h2><code>dorado</code> <code>basecaller</code></h2>
<ul>
<li><p>O padrão é salvar como <code>BAM</code>;</p>
<ul>
<li>Isso vai permitir salvar as modificações de base;</li>
</ul></li>
<li><p>A performance ideal requer GPU;</p>
<ul>
<li>Usar a tag <code>--device "cuda:all"</code>;</li>
</ul></li>
<li><p>Para gerar <code>fastq</code> usar <code>–emit-fastq</code>;</p></li>
<li><p>Para novos modelos é necessário utilizar <code>–emit-moves</code>;</p></li>
<li><p>É possível chamar e alinhar ao genoma de referência usando <code>--reference</code>;</p></li>
</ul>
</section>
<section id="dorado-basecaller-2" class="slide level2">
<h2><code>dorado</code> <code>basecaller</code></h2>
<div class="callout callout-note callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<p><strong>Tempo de processamento</strong></p>
</div>
<div class="callout-content">
<p>~ 12 horas de chamada de base + alinhamento utilizando GPU RTX 4060;</p>
<p>{model} precisa ser o caminho completo;</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb5-1"><a href=""></a><span class="ex">dorado</span> basecaller <span class="at">--recursive</span> <span class="at">--trim</span> <span class="st">"all"</span> <span class="at">--modified-bases</span> 5mCG_5hmCG 6mA <span class="at">--reference</span> {fasta} {modelo} pod5/ <span class="at">--device</span> <span class="st">"cuda:all"</span> <span class="op">&gt;</span> calls.bam</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</div>
</div>
</div>
</section>
<section id="controle-de-qualidade-qc" class="slide level2">
<h2>Controle de qualidade (QC)</h2>
</section>
<section id="controle-de-qualidade-qc-1" class="slide level2">
<h2>Controle de qualidade (QC)</h2>
<ul>
<li><p>Garantir que as leituras estejam adequadas para prosseguir as análises;</p></li>
<li><p><code>fastq</code>;</p></li>
<li><p>Similar a ideia para dados Illumina;</p></li>
</ul>
</section>
<section id="fastq" class="slide level2">
<h2>FASTQ</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://biocorecrg.github.io/RNAseq_course_2019/fastq.html"><img data-src="https://biocorecrg.github.io/RNAseq_course_2019/images/fastq_format.png" alt="Arquivo FASTQ."></a></p>
<figcaption>Arquivo FASTQ.</figcaption>
</figure>
</div>
</section>
<section id="phred-score" class="slide level2">
<h2>Phred score</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.drive5.com/usearch/manual/quality_score.html"><img data-src="https://www.drive5.com/usearch/manual/qscores.gif" class="quarto-figure quarto-figure-center" width="900" alt="Phred score."></a></p>
</figure>
</div>
<figcaption>Phred score.</figcaption>
</figure>
</div>
</section>
<section id="phred-score-1" class="slide level2">
<h2>Phred score</h2>
<p><span class="math display">\[
Q = -10\log_{10}{P}
\]</span></p>
<p><br></p>
<p><span class="math display">\[
P=10^{\frac{-Q}{10}}
\]</span></p>
</section>
<section id="phred-score-2" class="slide level2">
<h2>Phred score</h2>
<p><span class="math display">\[
P=10^{\frac{-Q}{10}}
\]</span></p>
<table class="caption-top">
<colgroup>
<col style="width: 20%">
<col style="width: 48%">
<col style="width: 30%">
</colgroup>
<thead>
<tr class="header">
<th>Q score</th>
<th>Probabilidade de chamada de base incorreta</th>
<th>Acurácia da chamada de base</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>10</td>
<td>1 em 10</td>
<td>90%</td>
</tr>
<tr class="even">
<td>20</td>
<td>1 em 100</td>
<td>99%</td>
</tr>
<tr class="odd">
<td>30</td>
<td>1 em 1000</td>
<td>99,9%</td>
</tr>
<tr class="even">
<td>40</td>
<td>1 em 10000</td>
<td>99,99%</td>
</tr>
</tbody>
</table>
</section>
<section id="qc" class="slide level2">
<h2>QC</h2>
<ul>
<li><p><code>FASTQC</code> tambem funciona;</p></li>
<li><p><code>NanoPack2</code> é a atual melhor opção;</p>
<ul>
<li><p><code>NanoPlot</code> para visualização das estatísticas;</p></li>
<li><p><code>Chopper</code> para trimagem;</p></li>
</ul></li>
</ul>
</section>
<section id="nanoplot" class="slide level2">
<h2>NanoPlot</h2>
<ul>
<li><p>Qualidade (<code>Q</code>);</p></li>
<li><p>Tamanho (<code>L</code>);</p></li>
<li><p>N50;</p></li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://hoytpr.github.io/bioinformatics-semester/materials/genomics-assembly-reporting/"><img data-src="https://hoytpr.github.io/bioinformatics-semester/fig/N50.png" class="quarto-figure quarto-figure-center" width="615"></a></p>
</figure>
</div>
</section>
<section id="nanoplot-1" class="slide level2">
<h2>NanoPlot</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb6-1"><a href=""></a><span class="ex">NanoPlot</span> <span class="at">-t</span> {threads} {--fastq/--bam} {files} <span class="at">-o</span> {OUTPUT_DIR_NAME}</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="chopper" class="slide level2">
<h2>Chopper</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb7-1"><a href=""></a><span class="ex">chopper</span> <span class="at">-q</span> 20 <span class="at">-l</span> 1000 reads.fastq <span class="op">&gt;</span> trimmed_reads.fastq </span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section></section>
<section>
<section id="mapeamento-de-sequências" class="title-slide slide level1 center">
<h1>Mapeamento de sequências</h1>

</section>
<section id="comando-minimap2" class="slide level2">
<h2>Comando <code>minimap2</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb8-1"><a href=""></a><span class="ex">minimap2</span> <span class="at">-ax</span> map-ont <span class="at">-Y</span> <span class="at">-t</span> {THREADS} <span class="at">-R</span> {RG} {REF} {fastq} <span class="at">-o</span> {sam}</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li><p><code>THREADS</code> = Número de threads;</p></li>
<li><p><code>RG</code> = Tag de registro da amostra;</p></li>
<li><p><code>REF</code> = Genoma de referência indexado.</p></li>
<li><p><code>fastq</code> = Arquivo das sequencias;</p></li>
<li><p><code>sam</code> = Arquivo do alinhamento no formato SAM;</p></li>
</ul>
</section></section>
<section>
<section id="chamada-de-variantes" class="title-slide slide level1 center">
<h1>Chamada de variantes</h1>

</section>
<section id="comando-freebayes" class="slide level2">
<h2>Comando <code>freebayes</code></h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode numberSource bash number-lines code-with-copy"><code class="sourceCode bash"><span id="cb9-1"><a href=""></a><span class="ex">freebayes</span> <span class="at">-f</span> {REF} <span class="at">-L</span> {BAM_LIST} <span class="at">-t</span> {REGIONS.BED} <span class="op">&gt;</span> var.vcf</span></code><button title="Copiar para a área de transferência" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="comando-clair3" class="slide level2">
<h2>Comando <code>clair3</code></h2>
</section></section>
<section id="variantes-estruturais" class="title-slide slide level1 center">
<h1>Variantes estruturais</h1>

</section>

<section id="determinação-de-haplótipos" class="title-slide slide level1 center">
<h1>Determinação de haplótipos</h1>

</section>

<section id="montagem-de-genomas" class="title-slide slide level1 center">
<h1>Montagem de genomas</h1>

</section>

<section id="modificação-de-bases" class="title-slide slide level1 center">
<h1>Modificação de bases</h1>

</section>

<section id="referências" class="title-slide slide level1 smaller scrollable">
<h1>Referências</h1>

<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-Begik2022" class="csl-entry" role="listitem">
Begik, Oguzhan, John S. Mattick, e Eva Maria Novoa. 2022. <span>“Exploring the Epitranscriptome by Native RNA Sequencing”</span>. <em>RNA</em> 28 (11): 1430–39. <a href="https://doi.org/10.1261/rna.079404.122">https://doi.org/10.1261/rna.079404.122</a>.
</div>
<div id="ref-gustafson2024" class="csl-entry" role="listitem">
Gustafson, Jonas A., Sophia B. Gibson, Nikhita Damaraju, Miranda P. G. Zalusky, Kendra Hoekzema, David Twesigomwe, Lei Yang, et al. 2024. <span>“High-Coverage Nanopore Sequencing of Samples from the 1000 Genomes Project to Build a Comprehensive Catalog of Human Genetic Variation”</span>. <em>Genome Research</em> 34 (11): 2061–73. <a href="https://doi.org/10.1101/gr.279273.124">https://doi.org/10.1101/gr.279273.124</a>.
</div>
<div id="ref-Jain2022" class="csl-entry" role="listitem">
Jain, Miten, Robin Abu-Shumays, Hugh E. Olsen, e Mark Akeson. 2022. <span>“Advances in Nanopore Direct RNA Sequencing”</span>. <em>Nature Methods</em> 19 (10): 1160–64. <a href="https://doi.org/10.1038/s41592-022-01633-w">https://doi.org/10.1038/s41592-022-01633-w</a>.
</div>
<div id="ref-Lu2025" class="csl-entry" role="listitem">
Lu, Chunzhe, Andrea Bonini, Jakob H. Viel, e Giovanni Maglia. 2025. <span>“Toward Single-Molecule Protein Sequencing Using Nanopores”</span>. <em>Nature Biotechnology</em> 43 (3): 312–22. <a href="https://doi.org/10.1038/s41587-025-02587-y">https://doi.org/10.1038/s41587-025-02587-y</a>.
</div>
<div id="ref-luo2024" class="csl-entry" role="listitem">
Luo, Yuan, Jiarong Zhang, Ming Ni, Zhusong Mei, Qiao Ye, Bingqian Guo, Longmei Fang, et al. 2024. <span>“Pilot Validation of on<span>-</span>Field STR Typing and Human Identity Testing by MinION Nanopore Sequencing”</span>. <em>ELECTROPHORESIS</em> 45 (9-10): 885–96. <a href="https://doi.org/10.1002/elps.202300234">https://doi.org/10.1002/elps.202300234</a>.
</div>
<div id="ref-nurk2022" class="csl-entry" role="listitem">
Nurk, Sergey, Sergey Koren, Arang Rhie, Mikko Rautiainen, Andrey V. Bzikadze, Alla Mikheenko, Mitchell R. Vollger, et al. 2022. <span>“The Complete Sequence of a Human Genome”</span>. <em>Science</em> 376 (6588): 44–53. <a href="https://doi.org/10.1126/science.abj6987">https://doi.org/10.1126/science.abj6987</a>.
</div>
<div id="ref-rang2018" class="csl-entry" role="listitem">
Rang, Franka J., Wigard P. Kloosterman, e Jeroen de Ridder. 2018. <span>“From Squiggle to Basepair: Computational Approaches for Improving Nanopore Sequencing Read Accuracy”</span>. <em>Genome Biology</em> 19 (1). <a href="https://doi.org/10.1186/s13059-018-1462-9">https://doi.org/10.1186/s13059-018-1462-9</a>.
</div>
<div id="ref-robinson2011" class="csl-entry" role="listitem">
Robinson, James T, Helga Thorvaldsdóttir, Wendy Winckler, Mitchell Guttman, Eric S Lander, Gad Getz, e Jill P Mesirov. 2011. <span>“Integrative Genomics Viewer”</span>. <em>Nature Biotechnology</em> 29 (1): 24–26. <a href="https://doi.org/10.1038/nbt.1754">https://doi.org/10.1038/nbt.1754</a>.
</div>
<div id="ref-smits2022" class="csl-entry" role="listitem">
Smits, Nathan, e Geoffrey J. Faulkner. 2022. <span>“Nanopore Sequencing to Identify Transposable Element Insertions and Their Epigenetic Modifications”</span>. Em, 151–71. Springer US. <a href="https://doi.org/10.1007/978-1-0716-2883-6_9">https://doi.org/10.1007/978-1-0716-2883-6_9</a>.
</div>
<div id="ref-wang2021" class="csl-entry" role="listitem">
Wang, Yunhao, Yue Zhao, Audrey Bollas, Yuru Wang, e Kin Fai Au. 2021. <span>“Nanopore Sequencing Technology, Bioinformatics and Applications”</span>. <em>Nature Biotechnology</em> 39 (11): 1348–65. <a href="https://doi.org/10.1038/s41587-021-01108-x">https://doi.org/10.1038/s41587-021-01108-x</a>.
</div>
<div id="ref-Workman2019" class="csl-entry" role="listitem">
Workman, Rachael E., Alison D. Tang, Paul S. Tang, Miten Jain, John R. Tyson, Roham Razaghi, Philip C. Zuzarte, et al. 2019. <span>“Nanopore Native RNA Sequencing of a Human Poly(A) Transcriptome”</span>. <em>Nature Methods</em> 16 (12): 1297–1305. <a href="https://doi.org/10.1038/s41592-019-0617-2">https://doi.org/10.1038/s41592-019-0617-2</a>.
</div>
</div>
</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<div class="footer footer-default">
<p>marcel.ferreira@unesp.br</p>
</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="nanopore_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="nanopore_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="nanopore_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"8\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'slow',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1500,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    

    <script>

      // htmlwidgets need to know to resize themselves when slides are shown/hidden.

      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current

      // slide changes (different for each slide format).

      (function () {

        // dispatch for htmlwidgets

        function fireSlideEnter() {

          const event = window.document.createEvent("Event");

          event.initEvent("slideenter", true, true);

          window.document.dispatchEvent(event);

        }

    

        function fireSlideChanged(previousSlide, currentSlide) {

          fireSlideEnter();

    

          // dispatch for shiny

          if (window.jQuery) {

            if (previousSlide) {

              window.jQuery(previousSlide).trigger("hidden");

            }

            if (currentSlide) {

              window.jQuery(currentSlide).trigger("shown");

            }

          }

        }

    

        // hookup for slidy

        if (window.w3c_slidy) {

          window.w3c_slidy.add_observer(function (slide_num) {

            // slide_num starts at position 1

            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);

          });

        }

    

      })();

    </script>

    

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copiada");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copiada");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp('/' + window.location.host + '/');
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>