<!DOCTYPE html>
<html lang="pt-BR"><head>
<script src="nanopore_files/libs/clipboard/clipboard.min.js"></script>
<script src="nanopore_files/libs/quarto-html/tabby.min.js"></script>
<script src="nanopore_files/libs/quarto-html/popper.min.js"></script>
<script src="nanopore_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="nanopore_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="nanopore_files/libs/quarto-html/light-border.css" rel="stylesheet">
<link href="nanopore_files/libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.6.39">

  <meta name="author" content="Marcel Ferreira">
  <meta name="dcterms.date" content="2025-07-25">
  <title>Sequenciamento de Terceira Gera√ß√£o com Oxford Nanopore</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="nanopore_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="nanopore_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
  <link rel="stylesheet" href="nanopore_files/libs/revealjs/dist/theme/quarto-423d414d36ce366a9d05f36d06304417.css">
  <link href="nanopore_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="nanopore_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Sequenciamento de Terceira Gera√ß√£o com Oxford Nanopore</h1>
  <p class="subtitle">Princ√≠pios e Aplica√ß√µes em Bioinform√°tica</p>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
Marcel Ferreira 
</div>
</div>
</div>

  <p class="date">2025-07-25</p>
</section>
<section>
<section id="sobre-mim" class="title-slide slide level1 center">
<h1>Sobre mim</h1>

</section>
<section id="muito-prazer.." class="slide level2">
<h2>Muito prazer..</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/marshall.jpg" width="420"></p>
<figcaption>Olar!</figcaption>
</figure>
</div>
</div><div class="column" style="font-size: 80%;">
<ul>
<li>Marcel;</li>
</ul>
<p><br></p>
<ul>
<li>F√≠sico m√©dico;</li>
</ul>
<p><br></p>
<ul>
<li>Me e PhD em Biotecnologia;</li>
</ul>
<p><br></p>
<ul>
<li>Desde 2023, p√≥s-doutorado em gen√©tica;</li>
</ul>
</div></div>
</section>
<section id="interesses" class="slide level2">
<h2>Interesses</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/marshall.jpg" width="420"></p>
<figcaption>Olar!</figcaption>
</figure>
</div>
</div><div class="column" style="font-size: 80%;">
<ul>
<li><p>M√©todos alternativos ao uso de animais;</p></li>
<li><p>Identific√£o humana via DNA;</p></li>
<li><p>Biologia do tecido √≥sseo;</p></li>
<li><p>Biomateriais;</p></li>
<li><p>Regenera√ß√£o tecidual;</p></li>
<li><p>Ci√™ncia aberta;</p></li>
</ul>
<p><br></p>
<div id="red-highlight" style="color: red; font-size: 140%; text-align: center; font-weight: bold;">
<p>Bioinform√°tica</p>
</div>
</div></div>
</section>
<section id="projeto" class="slide level2">
<h2>Projeto</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://isbet.org.br/diversidade-no-estagio-e-jovem-aprendiz/"><img data-src="https://isbet.org.br/wp-content/uploads/2024/08/grupo-de-jovens-multirraciais-sorrindo-para-a-camera-conceito-de-amizade-e-diversidade_442523-605.webp" class="quarto-figure quarto-figure-center" width="780" alt="Diversidade: como podemos utilizar DNA para predizer o fen√≥tipo?"></a></p>
</figure>
</div>
<figcaption>Diversidade: como podemos utilizar DNA para predizer o fen√≥tipo?</figcaption>
</figure>
</div>
</section>
<section id="n√£o-sou-da-nanopore" class="slide level2">
<h2>N√£o sou da nanopore</h2>
<div style="text-align: center;">
<iframe src="https://giphy.com/embed/zFa3vjySsETPa" width="414" height="480" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/zFa3vjySsETPa">via GIPHY</a>
</p>
</div>
</section></section>
<section>
<section id="sobre-voc√™s" class="title-slide slide level1 center">
<h1>Sobre voc√™s</h1>

</section>
<section id="sobre-voc√™s-1" class="slide level2">
<h2>Sobre voc√™s</h2>
<div style="text-align: center;">
<iframe src="https://giphy.com/embed/5wWf7H89PisM6An8UAU" width="960" height="558" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/editingandlayout-the-office-michael-scott-5wWf7H89PisM6An8UAU">via GIPHY</a>
</p>
</div>
</section></section>
<section>
<section id="sobre-essa-disciplina" class="title-slide slide level1 center">
<h1>Sobre essa disciplina</h1>

</section>
<section id="primeira-edi√ß√£o-de-muitas" class="slide level2">
<h2>Primeira edi√ß√£o (de muitas‚Ä¶üôè)</h2>
<ul>
<li>Desde maio estou como colaborador no programa de Gen√©tica do IBB;</li>
</ul>
<p><br></p>
<ul>
<li>Infelizmente n√£o ter√° pratica de bancada;</li>
</ul>
<p><br></p>
<ul>
<li>Vamos aprofundar sobre aspectos de bioinform√°tica das an√°lises;</li>
</ul>
<p><br></p>
<ul>
<li>Vamos focar em <strong><em>gen√¥mica</em></strong> e <strong><em>epigen√¥mica</em></strong>;</li>
</ul>
</section>
<section id="desafios" class="slide level2">
<h2>Desafios üöµ</h2>
<ul>
<li><p>Bancada..</p></li>
<li><p>Requisitos computacionais‚Ä¶</p></li>
<li><p>Tempo‚Ä¶</p></li>
<li><p>Programas‚Ä¶</p></li>
</ul>
</section>
<section id="material-online" class="slide level2">
<h2>Material online</h2>
<ul>
<li><a href="https://marceelrf.quarto.pub/nanopore_genetica">Esses slides</a>;</li>
</ul>
<p><br></p>
<ul>
<li><a href="https://marceelrf-material.netlify.app/nanopore.html">Livro de c√≥digos</a>;</li>
</ul>
</section>
<section id="fruto-do-meu-projeto" class="slide level2">
<h2>Fruto do meu projeto</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://workshopbioinfoforense.netlify.app/"><img data-src="images/Slide2.PNG" class="quarto-figure quarto-figure-center" width="462" alt="Workshop realizado em 2023 e 2024."></a></p>
</figure>
</div>
<figcaption>Workshop realizado em 2023 e 2024.</figcaption>
</figure>
</div>
</section>
<section id="m√©todo-do-ano-de-2022" class="slide level2">
<h2>M√©todo do ano de 2022</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.nature.com/articles/s41592-022-01730-w"><img data-src="methodoftheyear.png" class="quarto-figure quarto-figure-center" width="516" alt="Publicado na Nature:"></a></p>
</figure>
</div>
<figcaption>Publicado na Nature:</figcaption>
</figure>
</div>
</section>
<section id="passar-os-atalhos" class="slide level2">
<h2>Passar os atalhos</h2>
<div class="columns">
<div class="column" style="width:50%;">
<iframe src="https://giphy.com/embed/1n833bZxdzKzaErLe9" width="480" height="269" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/nba-expression-derrick-white-1n833bZxdzKzaErLe9">via GIPHY</a>
</p>
</div><div class="column" style="width:50%;">
<iframe src="https://giphy.com/embed/Cas0va26zqBvTO5vY9" width="480" height="480" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/pudgypenguins-rage-enough-ragequit-Cas0va26zqBvTO5vY9">via GIPHY</a>
</p>
</div></div>
</section>
<section id="bioinform√°tica" class="slide level2">
<h2>Bioinform√°tica</h2>
<ul>
<li>NIH: ‚Äú<em>Bioinformatics, as related to genetics and genomics, is a scientific subdiscipline that involves using computer technology to collect, store, analyze and disseminate biological data and information, such as DNA and amino acid sequences or annotations about those sequences.</em>‚Äù</li>
</ul>

<aside><div>
<p>NIH: National Institute of Health</p>
</div></aside></section>
<section id="bioinform√°tica-1" class="slide level2">
<h2>Bioinform√°tica</h2>
<ul>
<li>NIH: ‚Äú<em>Bioinformatics, as related to <strong>genetics</strong> and <strong>genomics</strong>, is a scientific subdiscipline that involves using computer technology to <strong>collect</strong>, <strong>store</strong>, <strong>analyze</strong> and <strong>disseminate</strong> biological data and information, such as <strong>DNA</strong> and <strong>amino acid sequences</strong> or annotations about those sequences.</em>‚Äù</li>
</ul>

<aside><div>
<p>NIH: National Institute of Health</p>
</div></aside></section>
<section id="bioinform√°tica-2" class="slide level2">
<h2>Bioinform√°tica</h2>
<ul>
<li>N√£o vamos focar nos algoritmos;</li>
</ul>
<p><br></p>
<ul>
<li>Mas sim nos dados produzidos e como analisa-los;</li>
</ul>
</section>
<section id="cronograma" class="slide level2">
<h2>Cronograma</h2>
<div class="columns">
<div class="column" style="font-size: 75%;">
<ul>
<li><p>Dia 1 - Manh√£:</p>
<ul>
<li><p>Apresenta√ß√£o;</p></li>
<li><p>Sequenciamento nanopore;</p></li>
<li><p>Instala√ß√£o dos softwares;</p></li>
</ul></li>
</ul>
<p><br></p>
<ul>
<li><p>Dia 1 - Tarde:</p>
<ul>
<li><p>Chamada de base;</p></li>
<li><p>Pr√°tica;</p></li>
</ul></li>
</ul>
<p><br></p>
</div><div class="column" style="font-size: 75%;">
<ul>
<li><p>Dia 2 - Manh√£:</p>
<ul>
<li><p>Mapeamento de sequ√™ncias;</p></li>
<li><p>Pr√°tica;</p></li>
</ul></li>
</ul>
<p><br></p>
<ul>
<li><p>Dia 2 - Tarde:</p>
<ul>
<li><p>Chamada de variantes;</p></li>
<li><p>Pr√°tica;</p></li>
</ul></li>
</ul>
</div></div>
</section>
<section id="cronograma-1" class="slide level2">
<h2>Cronograma</h2>
<div class="columns">
<div class="column" style="font-size: 75%;">
<ul>
<li><p>Dia 3 - Manh√£:</p>
<ul>
<li><p>Variantes estruturais;</p></li>
<li><p>Pr√°tica;</p></li>
</ul></li>
</ul>
<p><br></p>
<ul>
<li><p>Dia 3 - Tarde:</p>
<ul>
<li><p>Determina√ß√£o de hapl√≥tipos;</p></li>
<li><p>Pr√°tica;</p></li>
</ul></li>
</ul>
<p><br></p>
</div><div class="column" style="font-size: 75%;">
<ul>
<li><p>Dia 4 - Manh√£:</p>
<ul>
<li><p>Montagem de genomas;</p></li>
<li><p>Pr√°tica;</p></li>
</ul></li>
</ul>
<p><br></p>
<ul>
<li><p>Dia 4 - Tarde:</p>
<ul>
<li><p>Modifica√ß√£o de bases;</p></li>
<li><p>Pr√°tica;</p>
<p><br></p></li>
<li><p>Adeus üò•.</p></li>
</ul></li>
</ul>
</div></div>
</section>
<section id="avalia√ß√£o" class="slide level2" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Avalia√ß√£o</h2>
<ul>
<li>3 provas;</li>
<li>2 semin√°rios;</li>
<li>Estudo dirigido;</li>
</ul>
</section>
<section id="avalia√ß√£o-1" class="slide level2" data-auto-animate="true">
<h2 data-id="quarto-animate-title">Avalia√ß√£o</h2>
<ul>
<li><del>3 provas;</del></li>
<li><del>2 semin√°rios;</del></li>
<li><del>Estudo dirigido;</del></li>
</ul>
<p><br></p>
<ul>
<li>Participa√ß√£o na disciplina e fazer as atividades.</li>
</ul>
</section>
<section id="programas-e-dados" class="slide level2">
<h2>Programas e dados</h2>
<ul>
<li>Baixar arquivos do Drive [link];</li>
</ul>
<p><br></p>
<ul>
<li>Instalar o IGV <span class="citation" data-cites="robinson2011">(<a href="#/refer√™ncias" role="doc-biblioref" onclick="">Robinson et al. 2011</a>)</span> [<a href="https://igv.org/doc/desktop/">link</a>];</li>
</ul>
<p><br></p>
<ul>
<li>Usar o script de instala√ß√£o dos programas no Linux;</li>
</ul>
</section></section>
<section>
<section id="sequenciamento-via-ont" class="title-slide slide level1 center">
<h1>Sequenciamento via ONT</h1>

</section>
<section id="ordens-de-grandeza-do-genoma-humano" class="slide level2">
<h2>Ordens de Grandeza do Genoma Humano</h2>
<ul>
<li><p>N√∫mero de cromossomos: 46 (23 pares);</p></li>
<li><p>Tamanho do genoma haploide: ~3,2 bilh√µes de pares de bases (3,2 Gb);</p></li>
<li><p>N√∫mero estimado de genes codificadores de prote√≠nas: ~20.000;</p></li>
</ul>
</section>
<section id="section" class="slide level2">
<h2></h2>

<img data-src="humanchr.png" class="quarto-figure quarto-figure-center r-stretch"></section>
<section id="por-que-sequenciar" class="slide level2">
<h2>Por que sequenciar?</h2>
<div class="fragment">
<ul>
<li><p>A sequ√™ncia √© a ‚Äúreceita‚Äù da vida</p>
<ul>
<li>Ela determina a estrutura e fun√ß√£o das mol√©culas biol√≥gicas.</li>
</ul></li>
<li><p><strong>DNA</strong> ‚Üí <strong>RNA</strong> ‚Üí <strong>Prote√≠na</strong></p>
<ul>
<li>Altera√ß√µes na sequ√™ncia podem afetar a fun√ß√£o celular e causar doen√ßas.</li>
</ul></li>
<li><p>Entender a sequ√™ncia = entender o funcionamento dos organismos</p>
<ul>
<li>Do gene √† caracter√≠stica observ√°vel (fen√≥tipo).</li>
</ul></li>
<li><p>Permite identificar diferen√ßas gen√©ticas</p>
<ul>
<li>Entre indiv√≠duos, esp√©cies, popula√ß√µes ou c√©lulas (ex: c√¢ncer).</li>
</ul></li>
</ul>
</div>
</section>
<section id="gera√ß√£o-de-sequenciamento" class="slide level2">
<h2>Gera√ß√£o de sequenciamento</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://pit.edu.vn/en/post/generations-of-gene-sequencing-technology-139.htm"><img data-src="seqgens.png" alt="https://pit.edu.vn/en/post/generations-of-gene-sequencing-technology-139.htm"></a></p>
<figcaption>https://pit.edu.vn/en/post/generations-of-gene-sequencing-technology-139.htm</figcaption>
</figure>
</div>
</section>
<section id="sequenciamento-comparativo-entre-plataformas" class="slide level2">
<h2>Sequenciamento: Comparativo entre Plataformas</h2>
<p><br></p>
<div style="font-size:80%;">
<table class="caption-top" style="width:100%;">
<colgroup>
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
</colgroup>
<tbody>
<tr class="odd">
<td><strong>Plataforma</strong></td>
<td><strong>Tipo de Leitura</strong></td>
<td><strong>Tamanho de Leitura</strong></td>
<td><strong>Taxa de Erro</strong></td>
<td><strong>Tempo de Execu√ß√£o</strong></td>
<td><strong>Aplica√ß√µes Comuns</strong></td>
</tr>
<tr class="even">
<td>Illumina</td>
<td>Short reads</td>
<td>150‚Äì300 pares de bases</td>
<td>&lt;1%</td>
<td>1‚Äì2 dias</td>
<td>RNA-seq, exoma, WGS, genotipagem</td>
</tr>
<tr class="odd">
<td>PacBio Hifi</td>
<td>Long reads (alta fidelidade)</td>
<td>10‚Äì25 kb (HiFi)</td>
<td>~1%</td>
<td>1‚Äì2 dias</td>
<td>Montagem gen√¥mica, hapl√≥tipos, variantes estruturais</td>
</tr>
<tr class="even">
<td>Oxford Nanopre</td>
<td>Long/ultralong reads</td>
<td>10 kb ‚Äì &gt;1 Mb</td>
<td>5‚Äì10% (melhorando)</td>
<td>Horas a 2 dias</td>
<td>Metagen√¥mica, epigen√©tica, transcript√¥mica, forense</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="introdu√ß√£o-a-tecnologia-de-sequenciamento" class="slide level2">
<h2>Introdu√ß√£o a tecnologia de sequenciamento</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.genome.gov/geneticsglossary/DNASequencing#:~:text=DNA%20sequencing%20refers%20to%20the,use%20to%20develop%20and%20operate."><img data-src="seqmethods.png" class="quarto-figure quarto-figure-center" alt="https://www.genome.gov/geneticsglossary/DNASequencing#:~:text=DNA%20sequencing%20refers%20to%20the,use%20to%20develop%20and%20operate."></a></p>
</figure>
</div>
<figcaption>https://www.genome.gov/geneticsglossary/DNASequencing#:~:text=DNA%20sequencing%20refers%20to%20the,use%20to%20develop%20and%20operate.</figcaption>
</figure>
</div>
</section>
<section id="oxford-nanopore-technologies-ont" class="slide level2">
<h2>Oxford nanopore technologies (ONT)</h2>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-left">
<figure>
<p><img data-src="equipaments.png" class="quarto-figure quarto-figure-left"></p>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%;">
<ul>
<li><p>Port√°til e escal√°vel;</p></li>
<li><p>‚ÄúBarato‚Äù;</p></li>
<li><p>Aquisi√ß√£o de dados em tempo real;</p></li>
<li><p>Altos volumes de dados (fastq &gt; 50 Gb);</p></li>
<li><p>DNA* e RNA*;</p></li>
<li><p>Long reads (10 kb ‚Äì 100 Kb);</p></li>
<li><p>Ultra (100 Kb ‚Äì 300 Kb);</p></li>
<li><p>Recorde 4 Mb!!!!!!;</p></li>
<li><p>Acur√°cia atual de &gt;99%;</p></li>
</ul>
</div>
</div></div>
</div>
</section>
<section id="ont---flowcells" class="slide level2">
<h2>ONT - Flowcells</h2>

<img data-src="flowcells.png" class="r-stretch quarto-figure-center"><p class="caption">Valores te√≥rico de capacidade de sequenciamento.</p></section>
<section id="ont---princ√≠pio" class="slide level2">
<h2>ONT - Princ√≠pio</h2>

<img data-src="ont_principle.png" class="r-stretch quarto-figure-center"><p class="caption"><span class="citation" data-cites="wang2021">(<a href="#/refer√™ncias" role="doc-biblioref" onclick="">Wang et al. 2021</a>)</span></p></section>
<section id="ont---princ√≠pio-1" class="slide level2">
<h2>ONT - Princ√≠pio</h2>
<iframe data-external="1" src="https://www.youtube.com/embed/RcP85JHLmnI?si=tmBwQYEmgRZa34zA" width="90%" height="80%" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
</section>
<section id="ont---nanoporos" class="slide level2">
<h2>ONT - Nanoporos</h2>

<img data-src="napores.png" class="r-stretch quarto-figure-center"><p class="caption">Lu, C., Bonini, A., Viel, J.H.&nbsp;et al.&nbsp;Toward single-molecule protein sequencing using nanopores.&nbsp;Nat Biotechnol&nbsp;43, 312‚Äì322 (2025). <span class="citation" data-cites="Lu2025">(<a href="#/refer√™ncias" role="doc-biblioref" onclick="">Lu et al. 2025</a>)</span></p></section>
<section id="ont---tipos-de-leitura" class="slide level2">
<h2>ONT - Tipos de leitura</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/document/kit-14-device-and-informatics"><img data-src="duplex.webp" class="quarto-figure quarto-figure-center" width="900" alt="Duplex."></a></p>
</figure>
</div>
<figcaption>Duplex.</figcaption>
</figure>
</div>
</section>
<section id="ont---tipos-de-leitura-1" class="slide level2">
<h2>ONT - Tipos de leitura</h2>
<ul>
<li><p><code>Simplex</code>: O sequenciamento de uma √∫nica fita. A fita de DNA modelo passa pelo nanoporo e √© submetida ao basecalling. Isso √© realizado no MinKNOW.</p></li>
<li><p><code>Duplex</code>: O sequenciamento de ambas as fitas. A fita complementar √© lida imediatamente ap√≥s a fita modelo e o basecalling consensual para ambas as fitas leva a um aumento ainda maior na precis√£o. Isso √© realizado no Dorado.</p></li>
</ul>

<aside><div>
<p>MinKWON √© o software que controla o sequenciamento.</p>
</div></aside></section>
<section id="ont---leitura-da-fita" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="leituradafita.png"></p>
<figcaption>Sutton, M.A., Burton, A.S., Zaikova, E. et al.&nbsp;Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%">

</div>
</div></div>
</section>
<section id="ont---leitura-da-fita-1" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="leituradafita.png"></p>
<figcaption>Sutton, M.A., Burton, A.S., Zaikova, E. et al.&nbsp;Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%">
<ul>
<li><p>Leitura DNA = 400 bp por segundo;</p></li>
<li><p>RNA = 70 bp por segundo;</p></li>
<li><p>6 bases simult√¢neas por poro;</p></li>
<li><p>{A,T,C,G};</p></li>
</ul>
</div>
</div></div>
</section>
<section id="ont---leitura-da-fita-2" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="leituradafita.png"></p>
<figcaption>Sutton, M.A., Burton, A.S., Zaikova, E. et al.&nbsp;Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%">
<ul>
<li><p>Leitura DNA = 400 bp por segundo;</p></li>
<li><p>RNA = 70 bp por segundo;</p></li>
<li><p>6 bases simult√¢neas por poro;</p></li>
<li><p>{A,T,C,G};</p></li>
</ul>
<p><br></p>
<ul>
<li>4x4x4x4x4x4 =?</li>
</ul>
</div>
</div></div>
</section>
<section id="ont---leitura-da-fita-3" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="leituradafita.png"></p>
<figcaption>Sutton, M.A., Burton, A.S., Zaikova, E. et al.&nbsp;Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 80%">
<ul>
<li><p>Leitura DNA = 400 bp por segundo;</p></li>
<li><p>RNA = 70 bp por segundo;</p></li>
<li><p>6 bases simult√¢neas por poro;</p></li>
<li><p>{A,T,C,G};</p></li>
</ul>
<p><br></p>
<ul>
<li>4x4x4x4x4x4 =?</li>
</ul>
<p><br></p>
<ul>
<li>4096 combina√ß√µes!</li>
</ul>
</div>
</div></div>
</section>
<section id="ont---leitura-da-fita-4" class="slide level2">
<h2>ONT - Leitura da fita</h2>
<div class="center">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/technology/basecalling"><img data-src="https://a.storyblok.com/f/196663/5384x5511/5d1ab2a959/different-types-of-nanopore.png" width="488" alt="Modelos de poros tem dois centros de leitura."></a></p>
<figcaption>Modelos de poros tem dois centros de leitura.</figcaption>
</figure>
</div>
</div>
</section>
<section id="ont---rna" class="slide level2">
<h2>ONT - RNA</h2>

<img data-src="rna.png" class="quarto-figure quarto-figure-center r-stretch" width="1300"></section>
<section id="modifica√ß√µes-no-dna" class="slide level2">
<h2>Modifica√ß√µes no DNA</h2>
<div class="fragment">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="ont_prinmods.png" width="806"></p>
<figcaption>https://www.nature.com/articles/s41587-021-01108-x</figcaption>
</figure>
</div>
</div>
</section>
<section id="modifica√ß√µes-no-dna-1" class="slide level2">
<h2>Modifica√ß√µes no DNA</h2>
<ul>
<li>Existem cerca de 30 modifica√ß√µes de base descrita!</li>
</ul>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<p>{A, T, C, G, 5mC}</p>
</div><div class="column" style="width:50%;">
<p>{A, T, C, G, 5mC, 5hmC}</p>
</div></div>
</div>
</section>
<section id="modifica√ß√µes-no-dna-2" class="slide level2">
<h2>Modifica√ß√µes no DNA</h2>
<ul>
<li>Existem cerca de 30 modifica√ß√µes de base descrita!</li>
</ul>
<div class="columns">
<div class="column" style="width:50%;">
<p>{A, T, C, G, 5mC}</p>
<p><br></p>
<p>5x5x5x5x5x5 = 15625 combina√ß√µes!</p>
</div><div class="column" style="width:50%;">
<p>{A, T, C, G, 5mC, 5hmC}</p>
<p><br></p>
<p>6x6x6x6x6x6 = 46656 combina√ß√µes!</p>
</div></div>
</section>
<section id="basecallers" class="slide level2">
<h2>Basecallers</h2>

<img data-src="images/ont_geral.png" width="1200" class="r-stretch quarto-figure-center"><p class="caption">Basecalling.</p></section>
<section id="basecallers-1" class="slide level2">
<h2>Basecallers</h2>
<ul>
<li><p>Nanoporos diferentes cont√™m ‚Äúleitores‚Äù diferentes;</p>
<ul>
<li>R10.4.1 √© o modelo atual;</li>
</ul></li>
<li><p>Capturam o sinal el√©trico (POD5);</p></li>
<li><p>Basecallers transformam POD5 em FASTQ;</p></li>
<li><p>Utilizam Machine Learning (RNN);</p></li>
<li><p>Este processo pode ser feito em tempo real;</p></li>
</ul>

<aside><div>
<p>POD5 √© o formato atual, mas pode-se encontrar FAST5;</p>
</div></aside></section>
<section id="basecallers-2" class="slide level2">
<h2>Basecallers</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/technology/basecalling"><img data-src="https://a.storyblok.com/f/196663/809x716/e4743867e2/neural-networks.png" class="quarto-figure quarto-figure-center" width="565" alt="Como funcionam"></a></p>
</figure>
</div>
<figcaption>Como funcionam</figcaption>
</figure>
</div>
</section>
<section id="basecallers-3" class="slide level2">
<h2>Basecallers</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/technology/basecalling"><img data-src="https://a.storyblok.com/f/196663/2326x960/ae453e7f75/basecallers-timeline.png" class="quarto-figure quarto-figure-center" width="1200" alt="Evolu√ß√£o dos modelos."></a></p>
</figure>
</div>
<figcaption>Evolu√ß√£o dos modelos.</figcaption>
</figure>
</div>
</section>
<section id="basecallers-4" class="slide level2">
<h2>Basecallers</h2>
<p>Sugest√£o de leitura:</p>
<ul>
<li><em>‚Äú<strong>From squiggle to basepair: computational approaches for improving nanopore sequencing read accuracy</strong>‚Äù</em> <span class="citation" data-cites="rang2018">(<a href="#/refer√™ncias" role="doc-biblioref" onclick="">Rang, Kloosterman, e Ridder 2018</a>)</span>[<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1462-9">link</a>]</li>
</ul>
</section>
<section id="bancada" class="slide level2">
<h2>Bancada</h2>
<div class="fragment">
<ul>
<li>hmwDNA;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>Alta pureza;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>N√£o h√° necessidade de PCR;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li>N√£o h√° necessidade convers√µes por bissulfito de s√≥dio (e afins);</li>
</ul>
</div>
</section>
<section id="enriquecimento-do-sequenciamento" class="slide level2">
<h2>Enriquecimento do sequenciamento</h2>
<div class="fragment">
<ul>
<li><strong><em>Target</em></strong>;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li><strong><em>CRISPR</em></strong>;</li>
</ul>
</div>
<div class="fragment">
<ul>
<li><strong><em>Adaptive sampling</em></strong>;</li>
</ul>
</div>
</section>
<section id="enriquecimento-do-sequenciamento-1" class="slide level2">
<h2>Enriquecimento do sequenciamento</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1872497324001522"><img data-src="images/enrichment3.png" class="quarto-figure quarto-figure-center" width="1200" alt="Estrat√©gias de enriquecimento de ONT"></a></p>
</figure>
</div>
<figcaption>Estrat√©gias de enriquecimento de ONT</figcaption>
</figure>
</div>
</section>
<section id="potencial" class="slide level2">
<h2>Potencial</h2>
</section>
<section id="potencial---montagem-de-genomas" class="slide level2">
<h2>Potencial - montagem de genomas</h2>

<img data-src="montagemdegenomas.png" width="1000" class="r-stretch quarto-figure-center"><p class="caption">https://www.phgfoundation.org/briefing/clinical-long-read-sequencing</p></section>
<section id="potencial---montagem-de-genomas-1" class="slide level2">
<h2>Potencial - montagem de genomas</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.science.org/doi/10.1126/science.abj6987"><img data-src="completehumangenome.png" class="quarto-figure quarto-figure-center" width="596" alt="Montagem completa do genoma humano."></a></p>
</figure>
</div>
<figcaption>Montagem completa do genoma humano.</figcaption>
</figure>
</div>

<aside><div>
<p><span class="citation" data-cites="nurk2022">(<a href="#/refer√™ncias" role="doc-biblioref" onclick="">Nurk et al. 2022</a>)</span></p>
</div></aside></section>
<section id="potencial---montagem-de-genomas-2" class="slide level2">
<h2>Potencial - montagem de genomas</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.science.org/cms/10.1126/science.abj6987/asset/ec796ff1-289e-43cf-a065-ec6573199a8d/assets/images/large/science.abj6987-f1.jpg"><img data-src="completehumangenome_2.png" class="quarto-figure quarto-figure-center" width="393" alt="Aumento de bases no genoma."></a></p>
</figure>
</div>
<figcaption>Aumento de bases no genoma.</figcaption>
</figure>
</div>
</section>
<section id="potencial---hapl√≥tipos-longos" class="slide level2">
<h2>Potencial - hapl√≥tipos longos</h2>

<img data-src="longhap.png" width="1200" class="r-stretch quarto-figure-center"><p class="caption">Determinar hapl√≥tipos longos</p></section>
<section id="potencial---hapl√≥tipos-longos-1" class="slide level2">
<h2>Potencial - hapl√≥tipos longos</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.nature.com/articles/s41467-024-47349-7/figures/4"><img data-src="pecat.webp" class="quarto-figure quarto-figure-center" width="797" alt="Hapl√≥tipos do pai e da m√£e."></a></p>
</figure>
</div>
<figcaption>Hapl√≥tipos do pai e da m√£e.</figcaption>
</figure>
</div>
</section>
<section id="potencial---metila√ß√£o" class="slide level2">
<h2>Potencial - metila√ß√£o</h2>

<img data-src="hapmet.png" width="500" class="r-stretch quarto-figure-center"><p class="caption">Metila√ß√£o ao longo do gene MYEF2.</p></section>
<section id="potencial---outros" class="slide level2">
<h2>Potencial - outros</h2>
<ul>
<li>STR <span class="citation" data-cites="luo2024">(<a href="#/refer√™ncias" role="doc-biblioref" onclick="">Luo et al. 2024</a>)</span>;</li>
</ul>
<p><br></p>
<ul>
<li>Elementos transpon√≠veis <span class="citation" data-cites="smits2022">(<a href="#/refer√™ncias" role="doc-biblioref" onclick="">Smits e Faulkner 2022</a>)</span>;</li>
</ul>
<p><br></p>
<ul>
<li>Epitranscript√¥mica <span class="citation" data-cites="Workman2019 Begik2022 Jain2022">(<a href="#/refer√™ncias" role="doc-biblioref" onclick="">Workman et al. 2019</a>; <a href="#/refer√™ncias" role="doc-biblioref" onclick="">Begik, Mattick, e Novoa 2022</a>; <a href="#/refer√™ncias" role="doc-biblioref" onclick="">Jain et al. 2022</a>)</span>;</li>
</ul>
</section>
<section id="estado-atual" class="slide level2">
<h2>Estado atual</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/accuracy"><img data-src="https://a.storyblok.com/f/196663/10800x3600/86aa012b7e/20250613_dna_q-1.png" class="quarto-figure quarto-figure-center" width="1200" alt="Sequenciamento de DNA"></a></p>
</figure>
</div>
<figcaption>Sequenciamento de DNA</figcaption>
</figure>
</div>
</section>
<section id="estado-atual-1" class="slide level2">
<h2>Estado atual</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://nanoporetech.com/platform/accuracy"><img data-src="https://a.storyblok.com/f/196663/1158x952/4cc71d5637/base-modification-section-graph_mod.jpg" class="quarto-figure quarto-figure-center" width="608" alt="Metila√ß√£o."></a></p>
</figure>
</div>
<figcaption>Metila√ß√£o.</figcaption>
</figure>
</div>
</section>
<section id="futuro-da-tecnologia" class="slide level2">
<h2>Futuro da tecnologia</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://www.nature.com/articles/s41587-025-02587-y#citeas"><img data-src="protein.png" class="quarto-figure quarto-figure-center" alt="Lu, C., Bonini, A., Viel, J.H. et al.&nbsp;Toward single-molecule protein sequencing using nanopores. Nat Biotechnol 43, 312‚Äì322 (2025). https://doi.org/10.1038/s41587-025-02587-y"></a></p>
</figure>
</div>
<figcaption>Lu, C., Bonini, A., Viel, J.H. et al.&nbsp;Toward single-molecule protein sequencing using nanopores. Nat Biotechnol 43, 312‚Äì322 (2025). https://doi.org/10.1038/s41587-025-02587-y</figcaption>
</figure>
</div>
</section>
<section id="futuro-da-tecnologia-1" class="slide level2">
<h2>Futuro da tecnologia</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://comics.dcv.fct.unl.pt/resources/"><img data-src="multiomics.png" class="quarto-figure quarto-figure-center" alt="https://comics.dcv.fct.unl.pt/resources/"></a></p>
</figure>
</div>
<figcaption>https://comics.dcv.fct.unl.pt/resources/</figcaption>
</figure>
</div>
</section>
<section id="desafios-e-frustra√ß√µes.." class="slide level2">
<h2>Desafios (e frustra√ß√µes)..</h2>
<div style="text-align: center;">
<iframe src="https://giphy.com/embed/2zdVnqmhSfAvyQhzHj" width="480" height="480" style="" frameborder="0" class="giphy-embed" allowfullscreen="">
</iframe>
<p>
<a href="https://giphy.com/gifs/showtime-season-4-episode-2zdVnqmhSfAvyQhzHj">via GIPHY</a>
</p>
</div>
</section>
<section id="desafios-e-frustra√ß√µes..-1" class="slide level2">
<h2>Desafios (e frustra√ß√µes)..</h2>
<div class="columns">
<div class="column" style="width:50%;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><a href="https://github.com/rrwick"><img data-src="https://avatars.githubusercontent.com/u/7053555?v=4.png" width="418" alt="Ryan Wick"></a></p>
<figcaption>Ryan Wick</figcaption>
</figure>
</div>
</div><div class="column" style="width:50%;">
<div style="font-size: 90%">
<p><strong><em>‚ÄúMay 2025 update</em></strong></p>
<p><em>I‚Äôve recently heard that ONT is deprecating duplex basecalling ‚Äì not surprising given their recent silence on the topic. This is now the third time (after 2D and 1D<sup>2</sup>) that ONT has tried and dropped both-strand basecalling! So it seems that mixed simplex-duplex read sets like the ones in this post will end up a historical curiosity rather than a standard part of ONT sequencing.‚Äù</em> [<a href="https://rrwick.github.io/2024/05/08/duplex_assemblies.html">link</a>]</p>
</div>
</div></div>
</section>
<section id="vis√£o-sobre-a-tecnologia" class="slide level2">
<h2>Vis√£o sobre a tecnologia</h2>
</section>
<section id="vis√£o-sobre-a-tecnologia-1" class="slide level2" data-background-image="genildo.png" data-background-opacity=".25">
<h2>Vis√£o sobre a tecnologia</h2>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<div style="color: black; font-size: 140%; font-weight: bold; text-shadow: 2px 2px 4px black;">
<p><br></p>
<ul>
<li><p>Altera√ß√µes constantes;</p></li>
<li><p>Precisa sempre adaptar;</p></li>
<li><p>Muitas op√ß√µes de pipeline;</p></li>
<li><p>Diagn√≥stico üëé;</p></li>
</ul>
</div>
</div><div class="column" style="width:50%;">
<div style="color: black; font-size: 140%; font-weight: bold; text-shadow: 2px 2px 4px black;">
<p><br></p>
<ul>
<li><p>Melhorias constantes;</p></li>
<li><p>Somos pesquisadores;</p></li>
<li><p>Vers√°til;</p></li>
<li><p>Pesquisa üëç;</p></li>
</ul>
</div>
</div></div>
</div>
</section></section>
<section>
<section id="dados-para-essa-disciplina" class="title-slide slide level1 center">
<h1>Dados para essa disciplina</h1>

</section>
<section id="amostras" class="slide level2">
<h2>Amostras</h2>
</section>
<section id="genes" class="slide level2">
<h2>Genes</h2>
</section></section>
<section id="chamada-de-base-basecalling" class="title-slide slide level1 center">
<h1>Chamada de base (basecalling)</h1>

</section>

<section id="mapeamento-de-sequ√™ncias" class="title-slide slide level1 center">
<h1>Mapeamento de sequ√™ncias</h1>

</section>

<section id="chamada-de-variantes" class="title-slide slide level1 center">
<h1>Chamada de variantes</h1>

</section>

<section id="variantes-estruturais" class="title-slide slide level1 center">
<h1>Variantes estruturais</h1>

</section>

<section id="determina√ß√£o-de-hapl√≥tipos" class="title-slide slide level1 center">
<h1>Determina√ß√£o de hapl√≥tipos</h1>

</section>

<section id="montagem-de-genomas" class="title-slide slide level1 center">
<h1>Montagem de genomas</h1>

</section>

<section id="modifica√ß√£o-de-bases" class="title-slide slide level1 center">
<h1>Modifica√ß√£o de bases</h1>

</section>

<section id="refer√™ncias" class="title-slide slide level1 smaller scrollable">
<h1>Refer√™ncias</h1>

<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-Begik2022" class="csl-entry" role="listitem">
Begik, Oguzhan, John S. Mattick, e Eva Maria Novoa. 2022. <span>‚ÄúExploring the Epitranscriptome by Native RNA Sequencing‚Äù</span>. <em>RNA</em> 28 (11): 1430‚Äì39. <a href="https://doi.org/10.1261/rna.079404.122">https://doi.org/10.1261/rna.079404.122</a>.
</div>
<div id="ref-Jain2022" class="csl-entry" role="listitem">
Jain, Miten, Robin Abu-Shumays, Hugh E. Olsen, e Mark Akeson. 2022. <span>‚ÄúAdvances in Nanopore Direct RNA Sequencing‚Äù</span>. <em>Nature Methods</em> 19 (10): 1160‚Äì64. <a href="https://doi.org/10.1038/s41592-022-01633-w">https://doi.org/10.1038/s41592-022-01633-w</a>.
</div>
<div id="ref-Lu2025" class="csl-entry" role="listitem">
Lu, Chunzhe, Andrea Bonini, Jakob H. Viel, e Giovanni Maglia. 2025. <span>‚ÄúToward Single-Molecule Protein Sequencing Using Nanopores‚Äù</span>. <em>Nature Biotechnology</em> 43 (3): 312‚Äì22. <a href="https://doi.org/10.1038/s41587-025-02587-y">https://doi.org/10.1038/s41587-025-02587-y</a>.
</div>
<div id="ref-luo2024" class="csl-entry" role="listitem">
Luo, Yuan, Jiarong Zhang, Ming Ni, Zhusong Mei, Qiao Ye, Bingqian Guo, Longmei Fang, et al. 2024. <span>‚ÄúPilot Validation of on<span>-</span>Field STR Typing and Human Identity Testing by MinION Nanopore Sequencing‚Äù</span>. <em>ELECTROPHORESIS</em> 45 (9-10): 885‚Äì96. <a href="https://doi.org/10.1002/elps.202300234">https://doi.org/10.1002/elps.202300234</a>.
</div>
<div id="ref-nurk2022" class="csl-entry" role="listitem">
Nurk, Sergey, Sergey Koren, Arang Rhie, Mikko Rautiainen, Andrey V. Bzikadze, Alla Mikheenko, Mitchell R. Vollger, et al. 2022. <span>‚ÄúThe Complete Sequence of a Human Genome‚Äù</span>. <em>Science</em> 376 (6588): 44‚Äì53. <a href="https://doi.org/10.1126/science.abj6987">https://doi.org/10.1126/science.abj6987</a>.
</div>
<div id="ref-rang2018" class="csl-entry" role="listitem">
Rang, Franka J., Wigard P. Kloosterman, e Jeroen de Ridder. 2018. <span>‚ÄúFrom Squiggle to Basepair: Computational Approaches for Improving Nanopore Sequencing Read Accuracy‚Äù</span>. <em>Genome Biology</em> 19 (1). <a href="https://doi.org/10.1186/s13059-018-1462-9">https://doi.org/10.1186/s13059-018-1462-9</a>.
</div>
<div id="ref-robinson2011" class="csl-entry" role="listitem">
Robinson, James T, Helga Thorvaldsd√≥ttir, Wendy Winckler, Mitchell Guttman, Eric S Lander, Gad Getz, e Jill P Mesirov. 2011. <span>‚ÄúIntegrative Genomics Viewer‚Äù</span>. <em>Nature Biotechnology</em> 29 (1): 24‚Äì26. <a href="https://doi.org/10.1038/nbt.1754">https://doi.org/10.1038/nbt.1754</a>.
</div>
<div id="ref-smits2022" class="csl-entry" role="listitem">
Smits, Nathan, e Geoffrey J. Faulkner. 2022. <span>‚ÄúNanopore Sequencing to Identify Transposable Element Insertions and Their Epigenetic Modifications‚Äù</span>. Em, 151‚Äì71. Springer US. <a href="https://doi.org/10.1007/978-1-0716-2883-6_9">https://doi.org/10.1007/978-1-0716-2883-6_9</a>.
</div>
<div id="ref-wang2021" class="csl-entry" role="listitem">
Wang, Yunhao, Yue Zhao, Audrey Bollas, Yuru Wang, e Kin Fai Au. 2021. <span>‚ÄúNanopore Sequencing Technology, Bioinformatics and Applications‚Äù</span>. <em>Nature Biotechnology</em> 39 (11): 1348‚Äì65. <a href="https://doi.org/10.1038/s41587-021-01108-x">https://doi.org/10.1038/s41587-021-01108-x</a>.
</div>
<div id="ref-Workman2019" class="csl-entry" role="listitem">
Workman, Rachael E., Alison D. Tang, Paul S. Tang, Miten Jain, John R. Tyson, Roham Razaghi, Philip C. Zuzarte, et al. 2019. <span>‚ÄúNanopore Native RNA Sequencing of a Human Poly(A) Transcriptome‚Äù</span>. <em>Nature Methods</em> 16 (12): 1297‚Äì1305. <a href="https://doi.org/10.1038/s41592-019-0617-2">https://doi.org/10.1038/s41592-019-0617-2</a>.
</div>
</div>
</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<div class="footer footer-default">
<p>marcel.ferreira@unesp.br</p>
</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="nanopore_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="nanopore_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="nanopore_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="nanopore_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"8\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'fade',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'slow',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1500,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copiada");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copiada");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp('/' + window.location.host + '/');
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>