---
title: "Sequenciamento de Terceira Gera√ß√£o com Oxford Nanopore"
subtitle: "Princ√≠pios e Aplica√ß√µes em Bioinform√°tica"
author: "Marcel Ferreira"
date: today
format:
  revealjs:
    theme: simple
    transition: fade
    transition-speed: slow
    width: 1500
    page-layout: full
    reference-location: document
    chalkboard: true
    footer: "marcel.ferreira@unesp.br"
editor: visual
lang: pt-BR
slide-number: c/t
bibliography: references.bib
engine: knitr
---

# Sobre mim

## Muito prazer..

::: columns
::: {.column width="50%"}
![Olar!](images/marshall.jpg){fig-align="center" width="420"}
:::

::: {.column width="50%" style="font-size: 80%;"}
-   Marcel;

<br>

-   F√≠sico m√©dico;

<br>

-   Me e PhD em Biotecnologia;

<br>

-   Desde 2023, p√≥s-doutorado em gen√©tica;
:::
:::

## Interesses

::: columns
::: {.column width="50%"}
![Olar!](images/marshall.jpg){fig-align="center" width="420"}
:::

::: {.column width="50%" style="font-size: 80%;"}
-   M√©todos alternativos ao uso de animais;

-   Identific√£o humana via DNA;

-   Biologia do tecido √≥sseo;

-   Biomateriais;

-   Regenera√ß√£o tecidual;

-   Ci√™ncia aberta;

<br>

::: {#red-highlight style="color: red; font-size: 140%; text-align: center; font-weight: bold;"}
Bioinform√°tica
:::
:::
:::

## Projeto

[![Diversidade: como podemos utilizar DNA para predizer o fen√≥tipo?](https://isbet.org.br/wp-content/uploads/2024/08/grupo-de-jovens-multirraciais-sorrindo-para-a-camera-conceito-de-amizade-e-diversidade_442523-605.webp){fig-align="center" width="780"}](https://isbet.org.br/diversidade-no-estagio-e-jovem-aprendiz/)

## N√£o sou da nanopore

::: {style="text-align: center;"}
<iframe src="https://giphy.com/embed/zFa3vjySsETPa" width="414" height="480" style frameBorder="0" class="giphy-embed" allowFullScreen>

</iframe>

<p><a href="https://giphy.com/gifs/zFa3vjySsETPa">via GIPHY</a></p>
:::

# Sobre voc√™s

## Sobre voc√™s

::: {style="text-align: center;"}
<iframe src="https://giphy.com/embed/5wWf7H89PisM6An8UAU" width="960" height="558" style frameBorder="0" class="giphy-embed" allowFullScreen>

</iframe>

<p><a href="https://giphy.com/gifs/editingandlayout-the-office-michael-scott-5wWf7H89PisM6An8UAU">via GIPHY</a></p>
:::

# Sobre essa disciplina

## Primeira edi√ß√£o (de muitas...üôè)

-   Desde maio estou como colaborador no programa de Gen√©tica do IBB;

<br>

-   Infelizmente n√£o ter√° pratica de bancada;

<br>

-   Vamos aprofundar sobre aspectos de bioinform√°tica das an√°lises;

<br>

-   Vamos focar em ***gen√¥mica*** e ***epigen√¥mica***;

## Desafios üöµ

-   Bancada..

-   Requisitos computacionais...

-   Tempo...

-   Programas...

## Material online

-   [Esses slides](https://marceelrf.quarto.pub/nanopore_genetica);

<br>

-   [Livro de c√≥digos](https://marceelrf-material.netlify.app/nanopore.html);

## Fruto do meu projeto

[![Workshop realizado em 2023 e 2024.](images/Slide2.PNG){fig-align="center" width="462"}](https://workshopbioinfoforense.netlify.app/)

## M√©todo do ano de 2022

[![Publicado na Nature:](methodoftheyear.png){fig-align="center" width="516"}](https://www.nature.com/articles/s41592-022-01730-w)

## Passar os atalhos

::: columns
::: {.column width="50%"}
<iframe src="https://giphy.com/embed/1n833bZxdzKzaErLe9" width="480" height="269" style frameBorder="0" class="giphy-embed" allowFullScreen>

</iframe>

<p><a href="https://giphy.com/gifs/nba-expression-derrick-white-1n833bZxdzKzaErLe9">via GIPHY</a></p>
:::

::: {.column width="50%"}
<iframe src="https://giphy.com/embed/Cas0va26zqBvTO5vY9" width="480" height="480" style frameBorder="0" class="giphy-embed" allowFullScreen>

</iframe>

<p><a href="https://giphy.com/gifs/pudgypenguins-rage-enough-ragequit-Cas0va26zqBvTO5vY9">via GIPHY</a></p>
:::
:::

## Bioinform√°tica

-   NIH: "*Bioinformatics, as related to genetics and genomics, is a scientific subdiscipline that involves using computer technology to collect, store, analyze and disseminate biological data and information, such as DNA and amino acid sequences or annotations about those sequences.*"

::: aside
NIH: National Institute of Health
:::

## Bioinform√°tica

-   NIH: "*Bioinformatics, as related to **genetics** and **genomics**, is a scientific subdiscipline that involves using computer technology to **collect**, **store**, **analyze** and **disseminate** biological data and information, such as **DNA** and **amino acid sequences** or annotations about those sequences.*"

::: aside
NIH: National Institute of Health
:::

## Bioinform√°tica

-   N√£o vamos focar nos algoritmos;

<br>

-   Mas sim nos dados produzidos e como analisa-los;

## Cronograma

::: columns
::: {.column width="50%" style="font-size: 75%;"}
-   Dia 1 - Manh√£:

    -   Apresenta√ß√£o;

    -   Sequenciamento nanopore;

    -   Instala√ß√£o dos softwares;

<br>

-   Dia 1 - Tarde:

    -   Chamada de base;

    -   Pr√°tica;

<br>
:::

::: {.column width="50%" style="font-size: 75%;"}
-   Dia 2 - Manh√£:

    -   Mapeamento de sequ√™ncias;

    -   Pr√°tica;

<br>

-   Dia 2 - Tarde:

    -   Chamada de variantes;

    -   Pr√°tica;
:::
:::

## Cronograma

::: columns
::: {.column width="50%" style="font-size: 75%;"}
-   Dia 3 - Manh√£:

    -   Variantes estruturais;

    -   Pr√°tica;

<br>

-   Dia 3 - Tarde:

    -   Determina√ß√£o de hapl√≥tipos;

    -   Pr√°tica;

<br>
:::

::: {.column width="50%" style="font-size: 75%;"}
-   Dia 4 - Manh√£:

    -   Montagem de genomas;

    -   Pr√°tica;

<br>

-   Dia 4 - Tarde:

    -   Modifica√ß√£o de bases;

    -   Pr√°tica;

        <br>

    -   Adeus üò•.
:::
:::

## Avalia√ß√£o {auto-animate="true"}

-   3 provas;
-   2 semin√°rios;
-   Estudo dirigido;

------------------------------------------------------------------------

## Avalia√ß√£o {auto-animate="true"}

-   ~~3 provas;~~
-   ~~2 semin√°rios;~~
-   ~~Estudo dirigido;~~

<br>

-   Participa√ß√£o na disciplina e fazer as atividades.

## Programas e dados

-   Baixar arquivos do Drive \[link\];

<br>

-   Instalar o IGV [@robinson2011] \[[link](https://igv.org/doc/desktop/)\];

<br>

-   Usar o script de instala√ß√£o dos programas no Linux;

# Sequenciamento via ONT

## Ordens de Grandeza do Genoma Humano

-   N√∫mero de cromossomos: 46 (23 pares);

-   Tamanho do genoma haploide: \~3,2 bilh√µes de pares de bases (3,2 Gb);

-   N√∫mero estimado de genes codificadores de prote√≠nas: \~20.000;

## 

![](humanchr.png){fig-align="center"}

## Por que sequenciar?

. . .

-   A sequ√™ncia √© a "receita" da vida

    -   Ela determina a estrutura e fun√ß√£o das mol√©culas biol√≥gicas.

-   **DNA** ‚Üí **RNA** ‚Üí **Prote√≠na**

    -   Altera√ß√µes na sequ√™ncia podem afetar a fun√ß√£o celular e causar doen√ßas.

-   Entender a sequ√™ncia = entender o funcionamento dos organismos

    -   Do gene √† caracter√≠stica observ√°vel (fen√≥tipo).

-   Permite identificar diferen√ßas gen√©ticas

    -   Entre indiv√≠duos, esp√©cies, popula√ß√µes ou c√©lulas (ex: c√¢ncer).

## Gera√ß√£o de sequenciamento

[![https://pit.edu.vn/en/post/generations-of-gene-sequencing-technology-139.htm](seqgens.png)](https://pit.edu.vn/en/post/generations-of-gene-sequencing-technology-139.htm)

## Sequenciamento: Comparativo entre Plataformas

<br>

::: {style="font-size:80%;"}
|                |                              |                         |                     |                       |                                                      |
|------------|------------|------------|------------|------------|------------|
| **Plataforma** | **Tipo de Leitura**          | **Tamanho de Leitura**  | **Taxa de Erro**    | **Tempo de Execu√ß√£o** | **Aplica√ß√µes Comuns**                                |
| Illumina       | Short reads                  | 150--300 pares de bases | \<1%                | 1--2 dias             | RNA-seq, exoma, WGS, genotipagem                     |
| PacBio Hifi    | Long reads (alta fidelidade) | 10--25 kb (HiFi)        | \~1%                | 1--2 dias             | Montagem gen√¥mica, hapl√≥tipos, variantes estruturais |
| Oxford Nanopre | Long/ultralong reads         | 10 kb -- \>1 Mb         | 5--10% (melhorando) | Horas a 2 dias        | Metagen√¥mica, epigen√©tica, transcript√¥mica, forense  |
:::

## Introdu√ß√£o a tecnologia de sequenciamento

[![https://www.genome.gov/geneticsglossary/DNASequencing#:\~:text=DNA%20sequencing%20refers%20to%20the,use%20to%20develop%20and%20operate.](seqmethods.png){fig-align="center"}](https://www.genome.gov/geneticsglossary/DNASequencing#:~:text=DNA%20sequencing%20refers%20to%20the,use%20to%20develop%20and%20operate.)

## Oxford nanopore technologies (ONT)

. . .

::: columns
::: {.column width="50%"}
![](equipaments.png){fig-align="left"}
:::

::: {.column width="50%"}
::: {style="font-size: 80%;"}
-   Port√°til e escal√°vel;

-   "Barato";

-   Aquisi√ß√£o de dados em tempo real;

-   Altos volumes de dados (fastq \> 50 Gb);

-   DNA\* e RNA\*;

-   Long reads (10 kb -- 100 Kb);

-   Ultra (100 Kb -- 300 Kb);

-   Recorde 4 Mb!!!!!!;

-   Acur√°cia atual de \>99%;
:::
:::
:::

## ONT - Flowcells

![Valores te√≥rico de capacidade de sequenciamento.](flowcells.png){fig-align="center"}

## ONT - Princ√≠pio

![[@wang2021]](ont_principle.png){fig-align="center"}

## ONT - Princ√≠pio

{{< video https://youtu.be/RcP85JHLmnI?si=tmBwQYEmgRZa34zA width="90%" height="80%">}}

## ONT - Nanoporos

![Lu, C., Bonini, A., Viel, J.H.¬†et al.¬†Toward single-molecule protein sequencing using nanopores.¬†Nat Biotechnol¬†43, 312--322 (2025). [@Lu2025]](napores.png)

## ONT - Tipos de leitura

[![Duplex.](duplex.webp){fig-align="center" width="900"}](https://nanoporetech.com/document/kit-14-device-and-informatics)

## ONT - Tipos de leitura

-   `Simplex`: O sequenciamento de uma √∫nica fita. A fita de DNA modelo passa pelo nanoporo e √© submetida ao basecalling. Isso √© realizado no MinKNOW.

-   `Duplex`: O sequenciamento de ambas as fitas. A fita complementar √© lida imediatamente ap√≥s a fita modelo e o basecalling consensual para ambas as fitas leva a um aumento ainda maior na precis√£o. Isso √© realizado no Dorado.

::: aside
MinKNOW √© o software que controla o sequenciamento.
:::

## ONT - Leitura da fita

::: columns
::: {.column width="50%"}
![Sutton, M.A., Burton, A.S., Zaikova, E. et al. Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4](leituradafita.png)
:::

::: {.column width="50%"}
::: {style="font-size: 80%"}
:::
:::
:::

## ONT - Leitura da fita

::: columns
::: {.column width="50%"}
![Sutton, M.A., Burton, A.S., Zaikova, E. et al. Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4](leituradafita.png)
:::

::: {.column width="50%"}
::: {style="font-size: 80%"}
-   Leitura DNA = 400 bp por segundo;

-   RNA = 70 bp por segundo;

-   6 bases simult√¢neas por poro;

-   {A,T,C,G};
:::
:::
:::

## ONT - Leitura da fita

::: columns
::: {.column width="50%"}
![Sutton, M.A., Burton, A.S., Zaikova, E. et al. Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4](leituradafita.png)
:::

::: {.column width="50%"}
::: {style="font-size: 80%"}
-   Leitura DNA = 400 bp por segundo;

-   RNA = 70 bp por segundo;

-   6 bases simult√¢neas por poro;

-   {A,T,C,G};

<br>

-   4x4x4x4x4x4 =?
:::
:::
:::

## ONT - Leitura da fita

::: columns
::: {.column width="50%"}
![Sutton, M.A., Burton, A.S., Zaikova, E. et al. Radiation Tolerance of Nanopore Sequencing Technology for Life Detection on Mars and Europa. Sci Rep 9, 5370 (2019). https://doi.org/10.1038/s41598-019-41488-4](leituradafita.png)
:::

::: {.column width="50%"}
::: {style="font-size: 80%"}
-   Leitura DNA = 400 bp por segundo;

-   RNA = 70 bp por segundo;

-   6 bases simult√¢neas por poro;

-   {A,T,C,G};

<br>

-   4x4x4x4x4x4 =?

<br>

-   4096 combina√ß√µes!
:::
:::
:::

## ONT - Leitura da fita

::: center
[![Modelos de poros tem dois centros de leitura.](https://a.storyblok.com/f/196663/5384x5511/5d1ab2a959/different-types-of-nanopore.png){width="488"}](https://nanoporetech.com/platform/technology/basecalling)
:::

## ONT - RNA

![](rna.png){fig-align="center" width="1300"}

## Modifica√ß√µes no DNA

. . .

![https://www.nature.com/articles/s41587-021-01108-x](ont_prinmods.png){fig-align="center" width="806"}

## Modifica√ß√µes no DNA

-   Existem cerca de 30 modifica√ß√µes de base descrita!

. . .

::: columns
::: {.column width="50%"}
{A, T, C, G, 5mC}
:::

::: {.column width="50%"}
{A, T, C, G, 5mC, 5hmC}
:::
:::

## Modifica√ß√µes no DNA

-   Existem cerca de 30 modifica√ß√µes de base descrita!

::: columns
::: {.column width="50%"}
{A, T, C, G, 5mC}

<br>

5x5x5x5x5x5 = 15625 combina√ß√µes!
:::

::: {.column width="50%"}
{A, T, C, G, 5mC, 5hmC}

<br>

6x6x6x6x6x6 = 46656 combina√ß√µes!
:::
:::

## Basecallers

![Basecalling.](images/ont_geral.png){fig-align="center" width="1200"}

## Basecallers

-   Nanoporos diferentes cont√™m "leitores" diferentes;

    -   R10.4.1 √© o modelo atual;

-   Capturam o sinal el√©trico (POD5);

-   Basecallers transformam POD5 em FASTQ;

-   Utilizam Machine Learning (RNN);

-   Este processo pode ser feito em tempo real;

::: aside
POD5 √© o formato atual, mas pode-se encontrar FAST5;
:::

## Basecallers

[![Como funcionam](https://a.storyblok.com/f/196663/809x716/e4743867e2/neural-networks.png){fig-align="center" width="565"}](https://nanoporetech.com/platform/technology/basecalling)

## Basecallers

[![Evolu√ß√£o dos modelos.](https://a.storyblok.com/f/196663/2326x960/ae453e7f75/basecallers-timeline.png){fig-align="center" width="1200"}](https://nanoporetech.com/platform/technology/basecalling)

## Basecallers

Sugest√£o de leitura:

-   *"**From squiggle to basepair: computational approaches for improving nanopore sequencing read accuracy**"* [@rang2018]\[[link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1462-9)\]

## Bancada

. . .

-   hmwDNA;

. . .

-   Alta pureza;

. . .

-   N√£o h√° necessidade de PCR;

. . .

-   N√£o h√° necessidade convers√µes por bissulfito de s√≥dio (e afins);

## Enriquecimento do sequenciamento

. . .

-   ***Target***;

. . .

-   ***CRISPR***;

. . .

-   ***Adaptive sampling***;

## Enriquecimento do sequenciamento

[![Estrat√©gias de enriquecimento de ONT](images/enrichment3.png){fig-align="center" width="1200"}](https://www.sciencedirect.com/science/article/pii/S1872497324001522)

## Potencial

## Potencial - montagem de genomas

![https://www.phgfoundation.org/briefing/clinical-long-read-sequencing](montagemdegenomas.png){fig-align="center" width="1000"}

## Potencial - montagem de genomas

[![Montagem completa do genoma humano.](completehumangenome.png){fig-align="center" width="596"}](https://www.science.org/doi/10.1126/science.abj6987)

::: aside
[@nurk2022]
:::

## Potencial - montagem de genomas

[![Aumento de bases no genoma.](completehumangenome_2.png){fig-align="center" width="393"}](https://www.science.org/cms/10.1126/science.abj6987/asset/ec796ff1-289e-43cf-a065-ec6573199a8d/assets/images/large/science.abj6987-f1.jpg)

## Potencial - hapl√≥tipos longos

![Determinar hapl√≥tipos longos](longhap.png){fig-align="center" width="1200"}

## Potencial - hapl√≥tipos longos

[![Hapl√≥tipos do pai e da m√£e.](pecat.webp){fig-align="center" width="797"}](https://www.nature.com/articles/s41467-024-47349-7/figures/4)

## Potencial - metila√ß√£o

![Metila√ß√£o ao longo do gene MYEF2.](hapmet.png){fig-align="center" width="500"}

## Potencial - outros

-   STR [@luo2024];

<br>

-   Elementos transpon√≠veis [@smits2022];

<br>

-   Epitranscript√¥mica [@Workman2019; @Begik2022; @Jain2022];

## Estado atual

[![Sequenciamento de DNA](https://a.storyblok.com/f/196663/10800x3600/86aa012b7e/20250613_dna_q-1.png){fig-align="center" width="1200"}](https://nanoporetech.com/platform/accuracy)

$$
P = 10^{\frac{-Q}{10}}
$$

## Estado atual

[![Metila√ß√£o.](https://a.storyblok.com/f/196663/1158x952/4cc71d5637/base-modification-section-graph_mod.jpg){fig-align="center" width="608"}](https://nanoporetech.com/platform/accuracy)

## EPI2ME

::: columns
::: {.column width="50%"}
-   [EPI2ME](https://epi2me.nanoporetech.com/wfindex/) √© uma cole√ß√£o de pipelines fornecidos pela pr√≥pria ONT;

-   Nextflow;

-   √ötil para quem quer ou n√£o programar;
:::

::: {.column width="50%"}
[![](https://epi2me.nanoporetech.com/static/dbb47454a74ffdac6ab079de768cc8af/3421a/logo-dark.webp){fig-align="center" width="450"}](https://epi2me.nanoporetech.com/wfindex/)
:::
:::

## Futuro da tecnologia

[![Lu, C., Bonini, A., Viel, J.H. et al. Toward single-molecule protein sequencing using nanopores. Nat Biotechnol 43, 312--322 (2025). https://doi.org/10.1038/s41587-025-02587-y](protein.png){fig-align="center"}](https://www.nature.com/articles/s41587-025-02587-y#citeas)

## Futuro da tecnologia

[![https://comics.dcv.fct.unl.pt/resources/](multiomics.png){fig-align="center"}](https://comics.dcv.fct.unl.pt/resources/)

## Desafios (e frustra√ß√µes)..

::: {style="text-align: center;"}
<iframe src="https://giphy.com/embed/2zdVnqmhSfAvyQhzHj" width="480" height="480" style frameBorder="0" class="giphy-embed" allowFullScreen>

</iframe>

<p><a href="https://giphy.com/gifs/showtime-season-4-episode-2zdVnqmhSfAvyQhzHj">via GIPHY</a></p>
:::

## Desafios (e frustra√ß√µes)..

::: columns
::: {.column width="50%"}
[![Ryan Wick](https://avatars.githubusercontent.com/u/7053555?v=4){width="418"}](https://github.com/rrwick)
:::

::: {.column width="50%"}
::: {style="font-size: 90%"}
***"May 2025 update***

*I've recently heard that ONT is deprecating duplex basecalling -- not surprising given their recent silence on the topic. This is now the third time (after 2D and 1D^2^) that ONT has tried and dropped both-strand basecalling! So it seems that mixed simplex-duplex read sets like the ones in this post will end up a historical curiosity rather than a standard part of ONT sequencing."* \[[link](https://rrwick.github.io/2024/05/08/duplex_assemblies.html)\]
:::
:::
:::

## Vis√£o sobre a tecnologia

## Vis√£o sobre a tecnologia {background-image="genildo.png" background-opacity=".25"}

. . .

::: columns
::: {.column width="50%"}
::: {style="color: black; font-size: 140%; font-weight: bold; text-shadow: 2px 2px 4px black;"}
<br>

-   Altera√ß√µes constantes;

-   Precisa sempre adaptar;

-   Muitas op√ß√µes de pipeline;

-   Diagn√≥stico üëé;
:::
:::

::: {.column width="50%"}
::: {style="color: black; font-size: 140%; font-weight: bold; text-shadow: 2px 2px 4px black;"}
<br>

-   Melhorias constantes;

-   Somos pesquisadores;

-   Vers√°til;

-   Pesquisa üëç;
:::
:::
:::

# Dados para essa disciplina

## Amostras

-   As amostras deste disciplinas s√£o referentes ao projeto 1000 genomas [@gustafson2024]. \[[link](https://s3.amazonaws.com/1000g-ont/index.html?prefix=ALIGNMENT_AND_ASSEMBLY_DATA/100_PLUS/IN-HOUSE_MINIMAP2/)\]

-   Os c√≥digos das amostras s√£o:

    -   [HG00109](https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG00109);

    -   [HG00463](https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG00463&Product=DNA);

    -   [HG00663](https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG00663&Product=CC);

    -   [HG01063](https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG01063&PgId=166);

    -   [HG01303](https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG01303&Product=CC);

    -   [HG01524](https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=HG01524);

## Genes

```{r}
#| echo: false

library(gt)

readr::read_tsv(file = here::here("nanopore/genes_disciplina.bed"),
                col_names = c("chr","start","end","gene","strand")) |>
  dplyr::mutate(strand = ifelse(strand == 1,yes = "+", no = "-")) |>
  dplyr::arrange(chr, start) |>
  gt() |>
  tab_options(
    table.font.size = px(30),
    data_row.padding = px(12),
    table.width = pct(80)
  ) |>
  cols_align(
    align = "center",
    columns = everything()  # substitua pelos nomes reais das colunas
  )
```

## Programas

::: columns
::: {.column width="50%"}
-   Linux;

<br>

-   Usu√°rios windows podem usar WSL;

<br>

-   Script para instala√ß√£o;
:::

::: {.column width="50%"}
[![Conda](https://raw.githubusercontent.com/devicons/devicon/54cfe13ac10eaa1ef817a343ab0a9437eb3c2e08/icons/anaconda/anaconda-original-wordmark.svg){fig-align="center" width="350"}](https://anaconda.org/anaconda/conda)
:::
:::

## O que √© Conda?

::: {style="text-align: center;"}
<iframe src="https://giphy.com/embed/WdC26ECYnvTX4Mp2fI" width="480" height="480" style frameBorder="0" class="giphy-embed" allowFullScreen>

</iframe>

<p><a href="https://giphy.com/gifs/primevideo-lularich-WdC26ECYnvTX4Mp2fI">via GIPHY</a></p>
:::

## Conda

-   Gerenciador de pacotes/programas;

<br>

-   Cria ambiente reprodut√≠veis;

# Chamada de base (basecalling)

## Chamada de base

-   Procedimento no qual o sinal el√©trico (`.pod5`) √© convertido em bases (`.fastq` \| `.bam`);

-   Momento crucial onde se define o perfil da an√°lise que ser√° realizada;

-   Erros aqui prejudicam todas as demais an√°lises;

-   `Dorado`;

## Qual modelo usar?

```{bash eval=FALSE, echo=TRUE}
{analyte}_{pore}_{chemistry}_{speed}@version
```

-   `analyte`: DNA ou RNA?

-   `pore`: Qual t√©cnologia de poro? (`R10.4.1`/`R9.4.1`)

-   `chemistry`: Qual kit foi utilizado foi utilizado?

-   `speed`: Velocidade de leitura durante o sequenciamento (DNA 400bps)

-   `version`: Vers√£o do modelo

## Comando `dorado`

## Controle de qualidade (QC)

## NanoPlot

```{bash eval=FALSE, echo=TRUE}
NanoPlot -t {threads} {--fastq/--bam} {files} -o {OUTPUT_DIR_NAME}
```

## Chopper

```{bash eval=FALSE, echo=TRUE}
chopper -q 20 -l 1000 reads.fastq > trimmed_reads.fastq 
```

# Mapeamento de sequ√™ncias

## Comando `minimap2`

```{bash eval=FALSE, echo=TRUE}
minimap2 -ax map-ont -Y -t {THREADS} -R {RG} {REF} {fastq} -o {sam}
```

-   `THREADS` = N√∫mero de threads;

-   `RG` = Tag de registro da amostra;

-   `REF` = Genoma de refer√™ncia indexado.

-   `fastq` = Arquivo das sequencias;

-   `sam` = Arquivo do alinhamento no formato SAM;

# Chamada de variantes

## Comando `freebayes`

```{bash eval=FALSE, echo=TRUE}
freebayes -f {REF} -L {BAM_LIST} -t {REGIONS.BED} > var.vcf
```

## Comando `clair3`

# Variantes estruturais

# Determina√ß√£o de hapl√≥tipos

# Montagem de genomas

# Modifica√ß√£o de bases

# Refer√™ncias
